diff --git a/machines/provisioning/scripts/setup-symlinks/setup-symlinks.shell-script.shell.sh b/machines/provisioning/scripts/setup-symlinks/setup-symlinks.shell-script.shell.sh
old mode 100755
new mode 100644
index 1e2efa81663ad29c17bb0592216f62aa06f95df9..110c749779d581fa6ad0666452d5454ddc6670ce
--- a/machines/provisioning/scripts/setup-symlinks/setup-symlinks.shell-script.shell.sh
+++ b/machines/provisioning/scripts/setup-symlinks/setup-symlinks.shell-script.shell.sh
@@ -28,7 +28,6 @@ link "$FILES/git-global-ignore.provisioned-file.content.conf"  "$HOME/.config/gi
 link "$FILES/repos-editor-settings.provisioned-file.content.json" "$REPOS/.vscode/settings.json"
 
 echo "Setting up the launchers on PATH..."
-link "$AKASHA_ROOT/dotfiles/bin/ops"    "$HOME/.local/bin/ops"
 link "$AKASHA_ROOT/dotfiles/bin/akasha" "$HOME/.local/bin/akasha"
 link "$SCRIPTS/rg/rg.shell-script.shell.sh"                         "$HOME/.local/bin/rg"
 link "$SCRIPTS/find-session/find-session.shell-script.shell.sh"     "$HOME/.local/bin/find-session"
