runsWarrants: false
diff --git a/temper/temper-things/temper-thing.page-type.ts b/temper/temper-things/temper-thing.page-type.ts
index 253ebf05cebb72ed624fc3382f9692163c6a9474..676f5f621eefcae19e52a2c5a088476d96752c1c 100644
--- a/temper/temper-things/temper-thing.page-type.ts
+++ b/temper/temper-things/temper-thing.page-type.ts
@@ -43,15 +43,15 @@ export const temperThing = {
     "text-property/zone-name",
   ],
   properties: [
-    { pagePropertySlug: "key", required: false, many: false },
-    { pagePropertySlug: "icon", required: false, many: false },
-    { pagePropertySlug: "display-order", required: false, many: false },
-    { pagePropertySlug: "account-page", required: false, many: false },
-    { pagePropertySlug: "category-id", required: false, many: false },
-    { pagePropertySlug: "category", required: false, many: false },
-    { pagePropertySlug: "companion-id", required: false, many: false },
-    { pagePropertySlug: "eso-character-id", required: false, many: false },
-    { pagePropertySlug: "parent", required: false, many: false },
+    { pagePropertySlug: "text-property/key", required: false, many: false },
+    { pagePropertySlug: "text-property/icon", required: false, many: false },
+    { pagePropertySlug: "number-property/display-order", required: false, many: false },
+    { pagePropertySlug: "text-property/account-page", required: false, many: false },
+    { pagePropertySlug: "text-property/category-id", required: false, many: false },
+    { pagePropertySlug: "text-property/category", required: false, many: false },
+    { pagePropertySlug: "text-property/companion-id", required: false, many: false },
+    { pagePropertySlug: "text-property/eso-character-id", required: false, many: false },
+    { pagePropertySlug: "text-property/parent", required: false, many: false },
   ],
   invariants: [
     {
