diff --git a/pages/address/page-address.module.code.ts b/pages/address/page-address.module.code.ts
index 8c1dc093a4a72cd26f1e7769025b3f45528f9e2f..d5331859908237a49712a78a9cee8cdea05c85cc 100644
--- a/pages/address/page-address.module.code.ts
+++ b/pages/address/page-address.module.code.ts
@@ -1,4 +1,9 @@
 import { lowerUuid } from "../name-formats/pages/lower-uuid/lower-uuid.name-format.code.ts"
+import type { ById } from "../page-address-kinds/by-id/by-id.page-address-kind.code.ts"
+import type { InPageType } from "../page-address-kinds/in-page-type/in-page-type.page-address-kind.code.ts"
+import type { InPartOf } from "../page-address-kinds/in-part-of/in-part-of.page-address-kind.code.ts"
+
+export type PageAddress = ById | InPageType | InPartOf
 
 export type Address =
   | { readonly kind: "id"; readonly id: string }
diff --git a/pages/page-address-kinds/in-part-of/in-part-of.page-address-kind.code.ts b/pages/page-address-kinds/in-part-of/in-part-of.page-address-kind.code.ts
index 9d03067b163f06cfc781a59019a67810fcc74d60..ec151a3a2919891634b41bb331d1cb41042d2ebc 100644
--- a/pages/page-address-kinds/in-part-of/in-part-of.page-address-kind.code.ts
+++ b/pages/page-address-kinds/in-part-of/in-part-of.page-address-kind.code.ts
@@ -1,10 +1,9 @@
+import type { PageAddress } from "../../address/page-address.module.code.ts"
 import type { Slug } from "../../properties/slug.text-property.ts"
-import type { ById } from "../by-id/by-id.page-address-kind.code.ts"
-import type { InPageType } from "../in-page-type/in-page-type.page-address-kind.code.ts"
 
 export type InPartOf = {
   readonly pageTypeSlug: Slug
-  readonly partOf: ById | InPageType | InPartOf
+  readonly partOf: PageAddress
   readonly propertySlug: Slug
   readonly value: string
 }
