diff --git a/changes/refactor-changes/pages/rename-page-slug/rename-page.refactor-change.code.ts b/changes/refactor-changes/pages/rename-page-slug/rename-page.refactor-change.code.ts
index f299536c7ef431402b832c1c13343789d2ec2d99..342663169fa0c811a839ba50e5bfcbfcfe9912df 100644
--- a/changes/refactor-changes/pages/rename-page-slug/rename-page.refactor-change.code.ts
+++ b/changes/refactor-changes/pages/rename-page-slug/rename-page.refactor-change.code.ts
@@ -1,8 +1,9 @@
 import { dirname, join } from "node:path"
 import { parsedAs } from "@akasha/code/code-source"
-import { schemaOf } from "@akasha/indexes"
 import { besideAt } from "@akasha/pages/page-file-name"
 import { slugFor } from "@akasha/pages/page-property-key"
+import { propertiesIfNamedOf } from "@akasha/pages/page-type-properties"
+import { valueAt } from "@akasha/pages/page-value"
 import { renamePath } from "../../../atomic-changes/pages/rename-path/rename-path.atomic-change.code.ts"
 import { renameSlug } from "../../../atomic-changes/pages/rename-slug/rename-slug.atomic-change.code.ts"
 import { statedIn } from "../../../partial/pages/restate-value/restate-value.change-partial.code.ts"
