diff --git a/alan/tracking/to-dos/pages/brush-floss/brush-floss.to-do.ts b/alan/tracking/to-dos/pages/brush-floss/brush-floss.to-do.ts
index 9be7bdc26756419766718e63c44367a2e61d4a0f..13e55ab77037c3ea40581cf9eb99830b51751632 100644
--- a/alan/tracking/to-dos/pages/brush-floss/brush-floss.to-do.ts
+++ b/alan/tracking/to-dos/pages/brush-floss/brush-floss.to-do.ts
@@ -7,10 +7,10 @@ export const brushFloss = {
   title: "Brush + Floss",
   toDoCategory: "health",
   difficulty: "trivial",
-  toDoDueDate: "2026-09-07",
+  toDoDueDate: "2026-09-06",
   toDoPriority: "p3",
   toDoRecurrence: "FREQ=DAILY",
   toDoSortOrder: 49,
   toDoValueSlug: "health",
-  toDoLastCompletedAt: "2026-09-07T01:29:39.677Z",
+  toDoLastCompletedAt: "2026-08-18T23:35:30.981Z",
 } as const satisfies ToDo
