diff --git a/pages/views/pages/temper-tasks-completed.view.ts b/pages/views/pages/temper-tasks-completed.view.ts
index 1eaaa430fba5dd8e0bd116076a863c18a803b60b..9aaf47bfb539c77662780bb4c2c5caf2a40ca91a 100644
--- a/pages/views/pages/temper-tasks-completed.view.ts
+++ b/pages/views/pages/temper-tasks-completed.view.ts
@@ -6,7 +6,9 @@ export const temperTasksCompleted = {
   slug: "temper-tasks-completed",
   title: "Completed",
   navSlug: "temper-tasks",
+  pageType: "temper-task",
   viewPlace: 3,
+  narrows: [{ key: "completed-at", comparison: "empty", values: ["false"] }],
   viewSorts: [{ key: "updated-at", descending: true }],
   pageSize: 12,
   itemPageSize: 12,
