diff --git a/pages/views/pages/books-all-by-status.view.ts b/pages/views/pages/books-all-by-status.view.ts
index 05fed764567c16950dab0a1f4dcdf80aa74920c7..d9e3faa2329c32d5f31d2bab3c77661120876d55 100644
--- a/pages/views/pages/books-all-by-status.view.ts
+++ b/pages/views/pages/books-all-by-status.view.ts
@@ -6,6 +6,7 @@ export const booksAllByStatus = {
   slug: "books-all-by-status",
   title: "All by Status",
   navSlug: "books",
+  pageType: "ki-book",
   viewPlace: 3,
   layout: "cards",
   viewSorts: [{ key: "title", descending: false }],
diff --git a/pages/views/pages/books-completed.view.ts b/pages/views/pages/books-completed.view.ts
index 3e123b61aba12d6789092302e28df9313fde7ebe..35b27a927381f51f2ef0fcd7ffb56e958a83ae1f 100644
--- a/pages/views/pages/books-completed.view.ts
+++ b/pages/views/pages/books-completed.view.ts
@@ -6,6 +6,7 @@ export const booksCompleted = {
   slug: "books-completed",
   title: "Completed",
   navSlug: "books",
+  pageType: "ki-book",
   viewPlace: 2,
   layout: "cards",
   narrows: [{ key: "status", comparison: "is", values: ["Completed"] }],
diff --git a/pages/views/pages/books-reading.view.ts b/pages/views/pages/books-reading.view.ts
index 0e58638d0158dbf7364b3dc6280ba0e32858965d..4eb40436802827979ea5cef06e40faff5fffd6d9 100644
--- a/pages/views/pages/books-reading.view.ts
+++ b/pages/views/pages/books-reading.view.ts
@@ -6,6 +6,7 @@ export const booksReading = {
   slug: "books-reading",
   title: "Reading",
   navSlug: "books",
+  pageType: "ki-book",
   viewPlace: 0,
   layout: "cards",
   narrows: [{ key: "status", comparison: "in", values: ["In Progress", "Following"] }],
diff --git a/pages/views/pages/books-to-read.view.ts b/pages/views/pages/books-to-read.view.ts
index dfe4bbd4d42327cb41a2d5ba1115fd6137b9fcce..c6e7a2dfc04e7852e0202a35fd3508db217e3abb 100644
--- a/pages/views/pages/books-to-read.view.ts
+++ b/pages/views/pages/books-to-read.view.ts
@@ -6,6 +6,7 @@ export const booksToRead = {
   slug: "books-to-read",
   title: "To Read",
   navSlug: "books",
+  pageType: "ki-book",
   viewPlace: 1,
   layout: "cards",
   narrows: [{ key: "status", comparison: "is", values: ["Not Started"] }],
diff --git a/pages/views/pages/claude-accounts-claude-accounts.view.ts b/pages/views/pages/claude-accounts-claude-accounts.view.ts
index f0221404344db9fa83c04f83ea1f47468ea35854..d30a04759620843b011487849f04e11078b3d910 100644
--- a/pages/views/pages/claude-accounts-claude-accounts.view.ts
+++ b/pages/views/pages/claude-accounts-claude-accounts.view.ts
@@ -6,6 +6,7 @@ export const claudeAccountsClaudeAccounts = {
   slug: "claude-accounts-claude-accounts",
   title: "Claude Accounts",
   navSlug: "claude-accounts",
+  pageType: "claude-account",
   viewPlace: 0,
   viewSorts: [{ key: "seven-day-started-at", descending: false }],
   visibleProperties: [
