diff --git a/pages/views/pages/tasks-not-completed.view.ts b/pages/views/pages/tasks-not-completed.view.ts
index 309cfc29187357a86d8fba90c4ff0e231c3019f5..ce28d70fb06f55a95267c492ea673793e3ad5ead 100644
--- a/pages/views/pages/tasks-not-completed.view.ts
+++ b/pages/views/pages/tasks-not-completed.view.ts
@@ -9,10 +9,10 @@ export const tasksNotCompleted = {
   pageType: "to-do",
   viewPlace: 2,
   layout: "cards",
-  narrows: [{ key: "completed-at", comparison: "empty", values: ["true"] }],
-  viewSorts: [{ key: "created-at", descending: false }],
-  visibleProperties: ["due-date", "due-time", "link"],
-  hiddenPropertiesOrder: ["recurrence", "sort-order", "due-date", "category"],
+  narrows: [{ key: "to-do-completed-at", comparison: "empty", values: ["true"] }],
+  viewSorts: [{ key: "to-do-sort-order", descending: false }],
+  visibleProperties: ["to-do-due-date", "due-time", "link"],
+  hiddenPropertiesOrder: ["to-do-recurrence", "to-do-sort-order", "to-do-category"],
   pageSize: 12,
   itemPageSize: 12,
   groupPageSize: 6,
diff --git a/pages/views/pages/tasks-today.view.ts b/pages/views/pages/tasks-today.view.ts
index c2f637bc6ad1890b3611744b315d9e6ebe6f1250..42507a92086221f3c7a5eba23530fda6be5b9a7b 100644
--- a/pages/views/pages/tasks-today.view.ts
+++ b/pages/views/pages/tasks-today.view.ts
@@ -9,21 +9,21 @@ export const tasksToday = {
   pageType: "to-do",
   viewPlace: 0,
   layout: "cards",
-  narrows: [{ key: "due-date", comparison: "before", values: ["eso-day-next"] }],
+  narrows: [{ key: "to-do-due-date", comparison: "before", values: ["eso-day-next"] }],
   viewSorts: [
-    { key: "priority", descending: false },
+    { key: "to-do-priority", descending: false },
     { key: "title", descending: false },
   ],
   visibleProperties: [
-    "priority",
-    "due-date",
+    "to-do-priority",
+    "to-do-due-date",
     "due-time",
-    "value-slug",
+    "to-do-value-slug",
     "link",
     "difficulty",
-    "recurrence",
+    "to-do-recurrence",
   ],
-  hiddenPropertiesOrder: ["category", "sort-order"],
+  hiddenPropertiesOrder: ["to-do-category", "to-do-sort-order"],
   pageSize: 40,
   itemPageSize: 12,
   groupPageSize: 6,
diff --git a/pages/views/pages/tasks-up-next.view.ts b/pages/views/pages/tasks-up-next.view.ts
index 11c69063a25ea60afdc6101bd9639303ff2749a8..ddb62e0b4c229d7a5cdd21eb138a0a9d9a9998b0 100644
--- a/pages/views/pages/tasks-up-next.view.ts
+++ b/pages/views/pages/tasks-up-next.view.ts
@@ -9,12 +9,12 @@ export const tasksUpNext = {
   pageType: "to-do",
   viewPlace: 1,
   narrows: [
-    { key: "completed-at", comparison: "empty", values: ["true"] },
-    { key: "due-date", comparison: "at-or-after", values: ["eso-day-next"] },
+    { key: "to-do-completed-at", comparison: "empty", values: ["true"] },
+    { key: "to-do-due-date", comparison: "at-or-after", values: ["eso-day-next"] },
   ],
   viewSorts: [
-    { key: "due-date", descending: false },
+    { key: "to-do-due-date", descending: false },
     { key: "due-time", descending: false },
   ],
-  visibleProperties: ["priority", "due-date", "due-time"],
+  visibleProperties: ["to-do-priority", "to-do-due-date", "due-time"],
 } as const satisfies View
diff --git a/pages/views/pages/temper-tasks-today.view.ts b/pages/views/pages/temper-tasks-today.view.ts
index e09f8895e07e4de26033c251d8c7bdda8798c6be..2d164933b639b8b89877fff58877fc3ee8b293b2 100644
--- a/pages/views/pages/temper-tasks-today.view.ts
+++ b/pages/views/pages/temper-tasks-today.view.ts
@@ -8,10 +8,7 @@ export const temperTasksToday = {
   navSlug: "temper-tasks",
   pageType: "temper-task",
   viewPlace: 0,
-  narrows: [
-    { key: "completed-at", comparison: "empty", values: ["true"] },
-    { key: "due-date", comparison: "before", values: ["eso-day-next"] },
-  ],
+  narrows: [{ key: "due-date", comparison: "before", values: ["eso-day-next"] }],
   viewSorts: [
     { key: "priority", descending: false },
     { key: "character-sort-order", descending: true },
@@ -21,13 +18,12 @@ export const temperTasksToday = {
     "priority",
     "progress",
     "character",
-    "rrule",
+    "rrule-rule",
     "due-date",
     "due-time",
     "scope",
-    "link",
   ],
-  hiddenPropertiesOrder: ["page-type-id", "completion-card-id"],
+  hiddenPropertiesOrder: ["completion-card-id"],
   pageSize: 40,
   itemPageSize: 12,
   groupPageSize: 6,
diff --git a/pages/views/pages/temper-tasks-up-next.view.ts b/pages/views/pages/temper-tasks-up-next.view.ts
index 522c3406d1c5280ef238133e9de96a82ddaa193e..ed3236fe4bf1277c340a469808d8887efd65aecc 100644
--- a/pages/views/pages/temper-tasks-up-next.view.ts
+++ b/pages/views/pages/temper-tasks-up-next.view.ts
@@ -8,17 +8,13 @@ export const temperTasksUpNext = {
   navSlug: "temper-tasks",
   pageType: "temper-task",
   viewPlace: 1,
-  narrows: [
-    { key: "completed-at", comparison: "empty", values: ["true"] },
-    { key: "due-date", comparison: "at-or-after", values: ["eso-day-next"] },
-  ],
+  narrows: [{ key: "due-date", comparison: "at-or-after", values: ["eso-day-next"] }],
   viewSorts: [
     { key: "priority", descending: false },
     { key: "character-sort-order", descending: true },
     { key: "title", descending: false },
   ],
-  visibleProperties: ["priority", "progress", "character", "rrule", "due-date", "due-time"],
-  hiddenPropertiesOrder: ["page-type-id"],
+  visibleProperties: ["priority", "progress", "character", "rrule-rule", "due-date", "due-time"],
   pageSize: 12,
   itemPageSize: 12,
   groupPageSize: 6,
