diff --git a/pages/views/pages/tasks-today.view.ts b/pages/views/pages/tasks-today.view.ts
index 5d157e7cebb9f5d4105ad96fc9e3b74e25132b9a..c2f637bc6ad1890b3611744b315d9e6ebe6f1250 100644
--- a/pages/views/pages/tasks-today.view.ts
+++ b/pages/views/pages/tasks-today.view.ts
@@ -6,6 +6,7 @@ export const tasksToday = {
   slug: "tasks-today",
   title: "Today",
   navSlug: "tasks",
+  pageType: "to-do",
   viewPlace: 0,
   layout: "cards",
   narrows: [{ key: "due-date", comparison: "before", values: ["eso-day-next"] }],
