diff --git a/service-system/cluster-services/cluster-service.page-type.ts b/service-system/cluster-services/cluster-service.page-type.ts
index db526188cd7f2c2d52feec28a096af176c1bd6f7..755cd50dcf8bd47b45560fb6b25c3efc3be84a02 100644
--- a/service-system/cluster-services/cluster-service.page-type.ts
+++ b/service-system/cluster-services/cluster-service.page-type.ts
@@ -4,6 +4,7 @@ import type { ManifestCode } from "./properties/cluster-service-manifest-code.te
 import type { ClusterServiceSchedule } from "./properties/cluster-service-schedule.text-property.ts"
 import type { ContainerPort } from "./properties/container-port.number-property.ts"
 import type { Image } from "./properties/image.text-property.ts"
+import type { ManifestSlug } from "./properties/manifest-slug.relation-property.ts"
 import type { Namespace } from "./properties/namespace.text-property.ts"
 import type { Replicas } from "./properties/replicas.number-property.ts"
 import type { ResourceKind } from "./properties/resource-kind.text-property.ts"
