diff --git a/infrastructure/k8s-synth/synth-discovery/synth-discovery.module.code.ts b/infrastructure/k8s-synth/synth-discovery/synth-discovery.module.code.ts
index 84f05eaea5528d441a026fa6d97e7eede888e31b..44f067685135a8edbf7cbb2fb6467d25b17661d1 100644
--- a/infrastructure/k8s-synth/synth-discovery/synth-discovery.module.code.ts
+++ b/infrastructure/k8s-synth/synth-discovery/synth-discovery.module.code.ts
@@ -2,7 +2,6 @@ import { isAbsolute, join, relative } from "node:path"
 import { valuesOfType } from "@akasha/indexes"
 
 export const DISCOVERY_GLOBS: readonly string[] = [
-  "**/*.cluster-service.code.attachment.ts",
   "infrastructure/cluster-manifests/*-synth/*-synth.module.code.ts",
   "infrastructure/upscale/*-synth/*-synth.module.code.ts",
 ]
