runsWarrants: false
diff --git a/agents/agents b/agents/agents
deleted file mode 120000
index d5f58b529c22fa8bf8c330e4608ba495d8f8e6c7..0000000000000000000000000000000000000000
--- a/agents/agents
+++ /dev/null
@@ -1 +0,0 @@
-../../agents
\ No newline at end of file
diff --git a/alan/eso/eso-wallpapers/eso-wallpapers b/alan/eso/eso-wallpapers/eso-wallpapers
deleted file mode 120000
index 77e8b329b038aed6877d3deebb8a3f3c1f9b8253..0000000000000000000000000000000000000000
--- a/alan/eso/eso-wallpapers/eso-wallpapers
+++ /dev/null
@@ -1 +0,0 @@
-../../alan/eso/eso-wallpapers
\ No newline at end of file
diff --git a/alan/fitness/commands/exercise-commands b/alan/fitness/commands/exercise-commands
deleted file mode 120000
index 33f87403e797408d761b79d9ff1c10462ef76585..0000000000000000000000000000000000000000
--- a/alan/fitness/commands/exercise-commands
+++ /dev/null
@@ -1 +0,0 @@
-../../alan/fitness/commands
\ No newline at end of file
diff --git a/alan/fitness/exercise-access/exercise-access b/alan/fitness/exercise-access/exercise-access
deleted file mode 120000
index 4372f41da7d45948580fe5f23cf0ac0f7bd0bbb7..0000000000000000000000000000000000000000
--- a/alan/fitness/exercise-access/exercise-access
+++ /dev/null
@@ -1 +0,0 @@
-../../alan/fitness/exercise-access
\ No newline at end of file
