diff --git a/changes/workflow-templates/pages/workflow-gotrue/workflow-gotrue.workflow-template.declaration.ts b/changes/workflow-templates/pages/workflow-gotrue/workflow-gotrue.workflow-template.declaration.ts
index f64c0b8d16bce4a96e7ba54d14df36fc567ab274..7e6efa494d2f3d8940b7f78bfa43691227bd5787 100644
--- a/changes/workflow-templates/pages/workflow-gotrue/workflow-gotrue.workflow-template.declaration.ts
+++ b/changes/workflow-templates/pages/workflow-gotrue/workflow-gotrue.workflow-template.declaration.ts
@@ -32,6 +32,15 @@ export default workflow("gotrue", {
       dependsOn: ["gotrue-apply-namespace"],
     },
 
+    {
+      ...secretPlaceApply({
+        name: "gotrue-apply-secrets",
+        namespace: "gotrue",
+        resource: "gotrue-secrets",
+      }),
+      dependsOn: ["gotrue-apply-namespace"],
+    },
+
     {
       ...secretPlaceApply({
         name: "gotrue-apply-supabase-auth-admin-secrets",
@@ -139,7 +148,7 @@ export default workflow("gotrue", {
           kubernetes: { serviceAccountName: "pipeline-engine" },
         },
       }),
-      dependsOn: ["gotrue-ensure-auth-schema"],
+      dependsOn: ["gotrue-apply-secrets", "gotrue-ensure-auth-schema"],
     },
 
     {
@@ -157,6 +166,7 @@ export default workflow("gotrue", {
         },
       }),
       dependsOn: [
+        "gotrue-apply-secrets",
         "gotrue-apply-supabase-auth-admin-secrets",
         "gotrue-ensure-auth-schema",
         "gotrue-apply-manifests",
