diff --git a/pages/name-places/name-place.page-type.ts b/pages/name-places/name-place.page-type.ts
index 8fae0ab744ce839f8c3005591edc9b1a8b5433ce..6f67f520822b2e72456beca3ba5b4aa8607d74f4 100644
--- a/pages/name-places/name-place.page-type.ts
+++ b/pages/name-places/name-place.page-type.ts
@@ -1,6 +1,6 @@
 import type { Domain } from "@akasha/domains/domain"
-import type { NameFormatSlug } from "../page-properties/properties/name-format-slug.relation-property.ts"
 import type { PageType } from "../page-types/page-type.page-type.ts"
+import type { NameFormatSlug } from "../text-properties/properties/name-format-slug.relation-property.ts"
 
 export type NamePlace = Domain & {
   nameFormatSlug: NameFormatSlug | null
