diff --git a/pages/unique-kinds/pages/always.unique-kind.ts b/pages/unique-kinds/pages/always.unique-kind.ts
deleted file mode 100644
index 6d31cb53a8ff2b26fd2f3b360dffeb266846dd14..0000000000000000000000000000000000000000
--- a/pages/unique-kinds/pages/always.unique-kind.ts
+++ /dev/null
@@ -1,8 +0,0 @@
-import type { UniqueKind } from "../unique-kind.page-type.ts"
-
-export const always = {
-  id: "01a04edd-897d-7b88-90d8-c86522baad1d",
-  pageTypeSlug: "unique-kind",
-  slug: "always",
-  definition: "the value is unique across every page",
-} as const satisfies UniqueKind
