diff --git a/commands/domain/domain.command.test.ts b/commands/domain/domain.command.test.ts
index 90652ab9e2d7ff7e7517de2860f1e050311405cf..7b69f08716ac692f5d73e2f542bb132343f269b9 100644
--- a/commands/domain/domain.command.test.ts
+++ b/commands/domain/domain.command.test.ts
@@ -20,7 +20,7 @@ import {
 } from "./domain.command.code.ts"
 import { domain as domainCommand } from "./domain.command.ts"
 
-const ROOT = new URL("../../../", import.meta.url).pathname.replace(/\/$/, "")
+const ROOT = new URL("../../", import.meta.url).pathname.replace(/\/$/, "")
 
 const scratch = scratchWorld()
 
