diff --git a/commands/lint/lint.command.code.ts b/commands/lint/lint.command.code.ts
index eaafcabe0ac0877a9af8d39267fa625f0604c652..04c2449d72668c433727103ac606ecf1737fd6f6 100644
--- a/commands/lint/lint.command.code.ts
+++ b/commands/lint/lint.command.code.ts
@@ -2,7 +2,7 @@ import { resolve } from "node:path"
 import type { Found } from "@akasha/code-system/code-lint"
 import { lintedOver } from "@akasha/code-system/code-lint"
 import type { Answer, Given } from "../../command-system/calling/calling.module.code.ts"
-import { aiming, bounded } from "../../command-system/commands/test/test.command.code.ts"
+import { aiming, bounded } from "../test/test.command.code.ts"
 
 const FILE_PATH = "--file-path"
 
