diff --git a/temper/temper-eso-types/eso-timers/eso-timers.type-declaration.d.ts b/temper/temper-eso-types/eso-timers/eso-timers.type-declaration.d.ts
index 73e88f25941d4a2518370a4693c229db58b8ee11..bb34163429ada3b34ee815adf369d76c525b18aa 100644
--- a/temper/temper-eso-types/eso-timers/eso-timers.type-declaration.d.ts
+++ b/temper/temper-eso-types/eso-timers/eso-timers.type-declaration.d.ts
@@ -2,8 +2,8 @@ declare function setTimeout(this: void, fn: (this: void) => void, ms: number): n
 
 declare function setInterval(this: void, fn: (this: void) => void, ms: number): number
 
-declare const clearTimeout: (this: void, handle: number) => void
+declare function clearTimeout(this: void, handle: number): void
 
-declare const clearInterval: (this: void, handle: number) => void
+declare function clearInterval(this: void, handle: number): void
 
-declare const queueMicrotask: (this: void, fn: (this: void) => void) => void
+declare function queueMicrotask(this: void, fn: (this: void) => void): void
