runsWarrants: false
diff --git a/story/world-mechanics/world-quests/pages/heroic-quest.world-quest.ts b/story/world-mechanics/world-quests/pages/heroic-quest.world-quest.ts
deleted file mode 100644
index 8214833577b4ec9062329b0163822a0eb5480277..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-quests/pages/heroic-quest.world-quest.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldQuest } from "../world-quest.page-type.ts"
-
-export const heroicQuest = {
-  id: "01a0655a-0687-77e3-8d5a-163c517a554c",
-  pageTypeSlug: "world-quest",
-  slug: "heroic-quest",
-  title: "Heroic Quest",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldQuest
diff --git a/story/world-mechanics/world-quests/pages/heroic-quest/heroic-quest.world-quest.ts b/story/world-mechanics/world-quests/pages/heroic-quest/heroic-quest.world-quest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..ae76f2559c9b6b6015fff17029380e5f29f0bc55
--- /dev/null
+++ b/story/world-mechanics/world-quests/pages/heroic-quest/heroic-quest.world-quest.ts
@@ -0,0 +1,9 @@
+import type { WorldQuest } from "../../world-quest.page-type.ts"
+
+export const heroicQuest = {
+  id: "01a0655a-0687-77e3-8d5a-163c517a554c",
+  pageTypeSlug: "world-quest",
+  slug: "heroic-quest",
+  title: "Heroic Quest",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldQuest
diff --git a/story/world-mechanics/world-quests/pages/legendary-quest.world-quest.ts b/story/world-mechanics/world-quests/pages/legendary-quest.world-quest.ts
deleted file mode 100644
index d21682e68d8415ae9fefee887d725f79a3ec3d55..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-quests/pages/legendary-quest.world-quest.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldQuest } from "../world-quest.page-type.ts"
-
-export const legendaryQuest = {
-  id: "01a0655a-0688-72f0-8d16-874303f95a54",
-  pageTypeSlug: "world-quest",
-  slug: "legendary-quest",
-  title: "Legendary Quest",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldQuest
diff --git a/story/world-mechanics/world-quests/pages/legendary-quest/legendary-quest.world-quest.ts b/story/world-mechanics/world-quests/pages/legendary-quest/legendary-quest.world-quest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..182982e80d068cb7f63e553ef0b2a1b50245684d
--- /dev/null
+++ b/story/world-mechanics/world-quests/pages/legendary-quest/legendary-quest.world-quest.ts
@@ -0,0 +1,9 @@
+import type { WorldQuest } from "../../world-quest.page-type.ts"
+
+export const legendaryQuest = {
+  id: "01a0655a-0688-72f0-8d16-874303f95a54",
+  pageTypeSlug: "world-quest",
+  slug: "legendary-quest",
+  title: "Legendary Quest",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldQuest
diff --git a/story/world-mechanics/world-quests/pages/mythical-quest.world-quest.ts b/story/world-mechanics/world-quests/pages/mythical-quest.world-quest.ts
deleted file mode 100644
index c606a4ca398e24f477bf9010c020c31622b5a979..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-quests/pages/mythical-quest.world-quest.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldQuest } from "../world-quest.page-type.ts"
-
-export const mythicalQuest = {
-  id: "01a0655a-0688-764e-b679-2e8865598901",
-  pageTypeSlug: "world-quest",
-  slug: "mythical-quest",
-  title: "Mythical Quest",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldQuest
diff --git a/story/world-mechanics/world-quests/pages/mythical-quest/mythical-quest.world-quest.ts b/story/world-mechanics/world-quests/pages/mythical-quest/mythical-quest.world-quest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..01e83689649f98924bdc8e84524e3dbf041b078b
--- /dev/null
+++ b/story/world-mechanics/world-quests/pages/mythical-quest/mythical-quest.world-quest.ts
@@ -0,0 +1,9 @@
+import type { WorldQuest } from "../../world-quest.page-type.ts"
+
+export const mythicalQuest = {
+  id: "01a0655a-0688-764e-b679-2e8865598901",
+  pageTypeSlug: "world-quest",
+  slug: "mythical-quest",
+  title: "Mythical Quest",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldQuest
diff --git a/story/world-mechanics/world-quests/pages/royal-quest.world-quest.ts b/story/world-mechanics/world-quests/pages/royal-quest.world-quest.ts
deleted file mode 100644
index b2fa69dd7af693e1128c6ac2e6816da37f70ba36..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-quests/pages/royal-quest.world-quest.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldQuest } from "../world-quest.page-type.ts"
-
-export const royalQuest = {
-  id: "01a0655a-0688-73dd-b0d8-acefbdff8e3a",
-  pageTypeSlug: "world-quest",
-  slug: "royal-quest",
-  title: "Royal Quest",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldQuest
diff --git a/story/world-mechanics/world-quests/pages/royal-quest/royal-quest.world-quest.ts b/story/world-mechanics/world-quests/pages/royal-quest/royal-quest.world-quest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..3577ef2b6abee1eeb9b386208ef0aaa95282f80e
--- /dev/null
+++ b/story/world-mechanics/world-quests/pages/royal-quest/royal-quest.world-quest.ts
@@ -0,0 +1,9 @@
+import type { WorldQuest } from "../../world-quest.page-type.ts"
+
+export const royalQuest = {
+  id: "01a0655a-0688-73dd-b0d8-acefbdff8e3a",
+  pageTypeSlug: "world-quest",
+  slug: "royal-quest",
+  title: "Royal Quest",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldQuest
