runsWarrants: false
diff --git a/story/world-mechanics/world-recipes/pages/close-to-acid-jar.world-recipe.ts b/story/world-mechanics/world-recipes/pages/close-to-acid-jar.world-recipe.ts
deleted file mode 100644
index 622f07d6823833bb87c88877185535db8e1a2bd3..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-recipes/pages/close-to-acid-jar.world-recipe.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldRecipe } from "../world-recipe.page-type.ts"
-
-export const closeToAcidJar = {
-  id: "01a0655a-7b80-76b7-8a54-f55724e11b33",
-  pageTypeSlug: "world-recipe",
-  slug: "close-to-acid-jar",
-  title: "Close to Acid Jar",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/close-to-acid-jar/close-to-acid-jar.world-recipe.ts b/story/world-mechanics/world-recipes/pages/close-to-acid-jar/close-to-acid-jar.world-recipe.ts
new file mode 100644
index 0000000000000000000000000000000000000000..7a0ce52854f49af7c26f1f8e476cb5178a67aef6
--- /dev/null
+++ b/story/world-mechanics/world-recipes/pages/close-to-acid-jar/close-to-acid-jar.world-recipe.ts
@@ -0,0 +1,9 @@
+import type { WorldRecipe } from "../../world-recipe.page-type.ts"
+
+export const closeToAcidJar = {
+  id: "01a0655a-7b80-76b7-8a54-f55724e11b33",
+  pageTypeSlug: "world-recipe",
+  slug: "close-to-acid-jar",
+  title: "Close to Acid Jar",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/dragonpan-toast.world-recipe.ts b/story/world-mechanics/world-recipes/pages/dragonpan-toast.world-recipe.ts
deleted file mode 100644
index faf59d069f2f5ad98b361d63acfbd79a0f9a0391..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-recipes/pages/dragonpan-toast.world-recipe.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldRecipe } from "../world-recipe.page-type.ts"
-
-export const dragonpanToast = {
-  id: "01a0655a-7b80-7ca9-bb9e-76f26fcb9474",
-  pageTypeSlug: "world-recipe",
-  slug: "dragonpan-toast",
-  title: "Dragonpan Toast",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/dragonpan-toast/dragonpan-toast.world-recipe.ts b/story/world-mechanics/world-recipes/pages/dragonpan-toast/dragonpan-toast.world-recipe.ts
new file mode 100644
index 0000000000000000000000000000000000000000..03943d26e21f58618f7c6f18239649e5d0db3843
--- /dev/null
+++ b/story/world-mechanics/world-recipes/pages/dragonpan-toast/dragonpan-toast.world-recipe.ts
@@ -0,0 +1,9 @@
+import type { WorldRecipe } from "../../world-recipe.page-type.ts"
+
+export const dragonpanToast = {
+  id: "01a0655a-7b80-7ca9-bb9e-76f26fcb9474",
+  pageTypeSlug: "world-recipe",
+  slug: "dragonpan-toast",
+  title: "Dragonpan Toast",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/infused-dough-scaethen-dough.world-recipe.ts b/story/world-mechanics/world-recipes/pages/infused-dough-scaethen-dough.world-recipe.ts
deleted file mode 100644
index 9a301db9028fa9f32799c5f9d1920fa1b605e0ff..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-recipes/pages/infused-dough-scaethen-dough.world-recipe.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldRecipe } from "../world-recipe.page-type.ts"
-
-export const infusedDoughScaethenDough = {
-  id: "01a0655a-7b80-7197-b817-4c3a170e3ba6",
-  pageTypeSlug: "world-recipe",
-  slug: "infused-dough-scaethen-dough",
-  title: "Infused Dough (Scaethen Dough)",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/infused-dough-scaethen-dough/infused-dough-scaethen-dough.world-recipe.ts b/story/world-mechanics/world-recipes/pages/infused-dough-scaethen-dough/infused-dough-scaethen-dough.world-recipe.ts
new file mode 100644
index 0000000000000000000000000000000000000000..3888bbc625c94117e8143d82a52dafaaef39eb4e
--- /dev/null
+++ b/story/world-mechanics/world-recipes/pages/infused-dough-scaethen-dough/infused-dough-scaethen-dough.world-recipe.ts
@@ -0,0 +1,9 @@
+import type { WorldRecipe } from "../../world-recipe.page-type.ts"
+
+export const infusedDoughScaethenDough = {
+  id: "01a0655a-7b80-7197-b817-4c3a170e3ba6",
+  pageTypeSlug: "world-recipe",
+  slug: "infused-dough-scaethen-dough",
+  title: "Infused Dough (Scaethen Dough)",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/kraken-biscuit.world-recipe.ts b/story/world-mechanics/world-recipes/pages/kraken-biscuit.world-recipe.ts
deleted file mode 100644
index 16b3bfe7d11d6c1c12a12026fc285a7ff8efdad4..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-recipes/pages/kraken-biscuit.world-recipe.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldRecipe } from "../world-recipe.page-type.ts"
-
-export const krakenBiscuit = {
-  id: "01a0655a-7b80-7128-8388-54e9439d0ac1",
-  pageTypeSlug: "world-recipe",
-  slug: "kraken-biscuit",
-  title: "Kraken Biscuit",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/kraken-biscuit/kraken-biscuit.world-recipe.ts b/story/world-mechanics/world-recipes/pages/kraken-biscuit/kraken-biscuit.world-recipe.ts
new file mode 100644
index 0000000000000000000000000000000000000000..bab00f7b52a7ef31ac6c1ef7aad78ab596bc904f
--- /dev/null
+++ b/story/world-mechanics/world-recipes/pages/kraken-biscuit/kraken-biscuit.world-recipe.ts
@@ -0,0 +1,9 @@
+import type { WorldRecipe } from "../../world-recipe.page-type.ts"
+
+export const krakenBiscuit = {
+  id: "01a0655a-7b80-7128-8388-54e9439d0ac1",
+  pageTypeSlug: "world-recipe",
+  slug: "kraken-biscuit",
+  title: "Kraken Biscuit",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/poison-blood-oil.world-recipe.ts b/story/world-mechanics/world-recipes/pages/poison-blood-oil.world-recipe.ts
deleted file mode 100644
index 6c6c72466830c4d53da4a67cbf0a8fd3088ef9e0..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-recipes/pages/poison-blood-oil.world-recipe.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldRecipe } from "../world-recipe.page-type.ts"
-
-export const poisonBloodOil = {
-  id: "01a0655a-7b80-7eee-b83b-d183e23a4b63",
-  pageTypeSlug: "world-recipe",
-  slug: "poison-blood-oil",
-  title: "Poison Blood Oil",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/poison-blood-oil/poison-blood-oil.world-recipe.ts b/story/world-mechanics/world-recipes/pages/poison-blood-oil/poison-blood-oil.world-recipe.ts
new file mode 100644
index 0000000000000000000000000000000000000000..0631627887bcf24f48cfb44b26403100fd3accc3
--- /dev/null
+++ b/story/world-mechanics/world-recipes/pages/poison-blood-oil/poison-blood-oil.world-recipe.ts
@@ -0,0 +1,9 @@
+import type { WorldRecipe } from "../../world-recipe.page-type.ts"
+
+export const poisonBloodOil = {
+  id: "01a0655a-7b80-7eee-b83b-d183e23a4b63",
+  pageTypeSlug: "world-recipe",
+  slug: "poison-blood-oil",
+  title: "Poison Blood Oil",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/shadowloaf.world-recipe.ts b/story/world-mechanics/world-recipes/pages/shadowloaf.world-recipe.ts
deleted file mode 100644
index 8e58d0cd97fb0676654d211389fd426cda85f0f6..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-recipes/pages/shadowloaf.world-recipe.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldRecipe } from "../world-recipe.page-type.ts"
-
-export const shadowloaf = {
-  id: "01a0655a-7b80-7d74-a783-5fb2bc809ccd",
-  pageTypeSlug: "world-recipe",
-  slug: "shadowloaf",
-  title: "Shadowloaf",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/shadowloaf/shadowloaf.world-recipe.ts b/story/world-mechanics/world-recipes/pages/shadowloaf/shadowloaf.world-recipe.ts
new file mode 100644
index 0000000000000000000000000000000000000000..35de49d0148d32d8462f60606a227f92bbfa0030
--- /dev/null
+++ b/story/world-mechanics/world-recipes/pages/shadowloaf/shadowloaf.world-recipe.ts
@@ -0,0 +1,9 @@
+import type { WorldRecipe } from "../../world-recipe.page-type.ts"
+
+export const shadowloaf = {
+  id: "01a0655a-7b80-7d74-a783-5fb2bc809ccd",
+  pageTypeSlug: "world-recipe",
+  slug: "shadowloaf",
+  title: "Shadowloaf",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/stitch-bread.world-recipe.ts b/story/world-mechanics/world-recipes/pages/stitch-bread.world-recipe.ts
deleted file mode 100644
index 65e491e685cabf264eb4c20bcbedd88315422c95..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-recipes/pages/stitch-bread.world-recipe.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldRecipe } from "../world-recipe.page-type.ts"
-
-export const stitchBread = {
-  id: "01a0655a-7b80-7353-b851-1b4ebc73c378",
-  pageTypeSlug: "world-recipe",
-  slug: "stitch-bread",
-  title: "Stitch-bread",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/stitch-bread/stitch-bread.world-recipe.ts b/story/world-mechanics/world-recipes/pages/stitch-bread/stitch-bread.world-recipe.ts
new file mode 100644
index 0000000000000000000000000000000000000000..79d872b2e726fe13abc70eeaac970fdb2282ea62
--- /dev/null
+++ b/story/world-mechanics/world-recipes/pages/stitch-bread/stitch-bread.world-recipe.ts
@@ -0,0 +1,9 @@
+import type { WorldRecipe } from "../../world-recipe.page-type.ts"
+
+export const stitchBread = {
+  id: "01a0655a-7b80-7353-b851-1b4ebc73c378",
+  pageTypeSlug: "world-recipe",
+  slug: "stitch-bread",
+  title: "Stitch-bread",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/yuck-bomb.world-recipe.ts b/story/world-mechanics/world-recipes/pages/yuck-bomb.world-recipe.ts
deleted file mode 100644
index ca6379827c56ee478562fb5ab143eb6910db0f5e..0000000000000000000000000000000000000000
--- a/story/world-mechanics/world-recipes/pages/yuck-bomb.world-recipe.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-import type { WorldRecipe } from "../world-recipe.page-type.ts"
-
-export const yuckBomb = {
-  id: "01a0655a-7b80-752a-9391-00e8523f7e60",
-  pageTypeSlug: "world-recipe",
-  slug: "yuck-bomb",
-  title: "Yuck Bomb",
-  worldSlug: "the-wandering-inn",
-} as const satisfies WorldRecipe
diff --git a/story/world-mechanics/world-recipes/pages/yuck-bomb/yuck-bomb.world-recipe.ts b/story/world-mechanics/world-recipes/pages/yuck-bomb/yuck-bomb.world-recipe.ts
new file mode 100644
index 0000000000000000000000000000000000000000..1c71691e4be97cebc2f4aeeb8362bca78372674d
--- /dev/null
+++ b/story/world-mechanics/world-recipes/pages/yuck-bomb/yuck-bomb.world-recipe.ts
@@ -0,0 +1,9 @@
+import type { WorldRecipe } from "../../world-recipe.page-type.ts"
+
+export const yuckBomb = {
+  id: "01a0655a-7b80-752a-9391-00e8523f7e60",
+  pageTypeSlug: "world-recipe",
+  slug: "yuck-bomb",
+  title: "Yuck Bomb",
+  worldSlug: "the-wandering-inn",
+} as const satisfies WorldRecipe
