diff --git a/command-system/address-mapping/address-mapping.module.code.ts b/command-system/address-mapping/address-mapping.module.code.ts
index 5c994697058b39a53a4ef77679e90fd99016fb14..eef400b53b5567e6e79d9bd112e3c605b3770ba4 100644
--- a/command-system/address-mapping/address-mapping.module.code.ts
+++ b/command-system/address-mapping/address-mapping.module.code.ts
@@ -10,7 +10,7 @@ import { textIn } from "../worked-typing/worked-typing.module.code.ts"
 
 const RUNNER = "change-runner"
 
-const REACHED = ["change-partial", "change-command"]
+const REACHED = ["change", "change-partial", "change-command"]
 
 const ADDRESSED = "addressed"
 
