diff --git a/checks/code-checks/pages/relation-resolves/relation-resolves.code-check.ts b/checks/code-checks/pages/relation-resolves/relation-resolves.code-check.ts
index 9fbe4bdf6b5f816209c8f6a788c413f31b0e293f..3eeb019175df757b3caf987103f7587abe4bb3c5 100644
--- a/checks/code-checks/pages/relation-resolves/relation-resolves.code-check.ts
+++ b/checks/code-checks/pages/relation-resolves/relation-resolves.code-check.ts
@@ -9,7 +9,7 @@ export const relationResolves = {
   code: "ts",
   test: "ts",
   testFixtures: "ts",
-  runsOnPatch: false,
+  runsOnPatch: true,
   runsOnWorktree: false,
   runsOnDeploy: false,
   runsOnAudit: false,
