writerOwesReading: false
diff --git a/changes/modules/change-answer/change-answer.module.types.ts b/changes/modules/change-answer/change-answer.module.types.ts
index 338832ea629d728e7bff62617f6c1d41c1476c88..eae5b0a823ff2254e531eacbcb1aeb13966fd5eb 100644
--- a/changes/modules/change-answer/change-answer.module.types.ts
+++ b/changes/modules/change-answer/change-answer.module.types.ts
@@ -41,3 +41,8 @@ export type Answer = {
   readonly edits: readonly Edit[]
   readonly refused: string | null
 }
+
+export type Said = {
+  readonly edits: readonly (Edit | Stated)[]
+  readonly refused: string | null
+}
