diff --git a/pages/page.page-type.ts b/pages/page.page-type.ts
index 0747a1baf3c57528f439c610bdd945ae248d71ba..b53f970e81cca9d65c7796fa1a54fd7d6711a90d 100644
--- a/pages/page.page-type.ts
+++ b/pages/page.page-type.ts
@@ -1,5 +1,6 @@
 import type { PageType } from "./page-types/page-type.page-type.ts"
 import type { Cover } from "./properties/cover.text-property.ts"
+import type { Description } from "./properties/description.text-property.ts"
 import type { Id } from "./properties/id.text-property.ts"
 import type { PageTypeSlug } from "./properties/page-type-slug.relation-property.ts"
 import type { Slug } from "./properties/slug.text-property.ts"
@@ -10,6 +11,7 @@ export type Page = {
   pageTypeSlug: PageTypeSlug
   slug: Slug
   title?: Title
+  description?: Description
   cover?: Cover
 }
 
@@ -25,11 +27,13 @@ export const page = {
     { pagePropertySlug: "page-type-slug", required: true, many: false },
     { pagePropertySlug: "slug", required: true, many: false },
     { pagePropertySlug: "title", required: false, many: false },
+    { pagePropertySlug: "description", required: false, many: false },
     { pagePropertySlug: "cover", required: false, many: false },
   ],
   partSlugs: [
     "relation-property/page-type-slug",
     "text-property/cover",
+    "text-property/description",
     "text-property/id",
     "text-property/slug",
     "text-property/title",
diff --git a/pages/pages.workspace-package.ts b/pages/pages.workspace-package.ts
index 57475c4b2813f5cac0cf0d0cb33862717194f939..9cd991f35e95af50cac5e6264359592cdf41f1cb 100644
--- a/pages/pages.workspace-package.ts
+++ b/pages/pages.workspace-package.ts
@@ -53,7 +53,6 @@ export const pages = {
     "page-type/generator-kind",
     "page-type/unique-kind",
     "page-type/page-query",
-    "text-property/description",
     "workspace-package/indexes",
     "workspace-package/pages-query",
     "workspace-package/pages-system-service",
diff --git a/temper/temper-things/temper-thing.page-type.ts b/temper/temper-things/temper-thing.page-type.ts
index 283c13e76084a5da336611384227449f0d970bed..5582c178ec693ef14aee4a0305561e438057d100 100644
--- a/temper/temper-things/temper-thing.page-type.ts
+++ b/temper/temper-things/temper-thing.page-type.ts
@@ -1,6 +1,5 @@
 import type { Page } from "@akasha/pages-system/page"
 import type { PageType } from "@akasha/pages-system/page-type"
-import type { Description } from "../../pages/properties/description.text-property.ts"
 import type { Title } from "../../pages/properties/title.text-property.ts"
 import type { AccountPage } from "./properties/account-page.text-property.ts"
 import type { Category } from "./properties/category.text-property.ts"
@@ -15,7 +14,6 @@ import type { Parent } from "./properties/parent.text-property.ts"
 export type TemperThing = Page & {
   title: Title
   key?: Key
-  description?: Description
   icon?: Icon
   displayOrder?: DisplayOrder
   accountPage?: AccountPage
@@ -49,7 +47,6 @@ export const temperThing = {
   properties: [
     { pagePropertySlug: "title", required: true, many: false },
     { pagePropertySlug: "key", required: false, many: false },
-    { pagePropertySlug: "description", required: false, many: false },
     { pagePropertySlug: "icon", required: false, many: false },
     { pagePropertySlug: "display-order", required: false, many: false },
     { pagePropertySlug: "account-page", required: false, many: false },
