diff --git a/pages/queries/pages/alan-harness-tracking-field-all.page-query.ts b/pages/queries/pages/alan-harness-tracking-field-all.page-query.ts
index 7263933e154496ae554b50822bf38896c69a78e8..84892ea7787e5fe19074f228bf65cab0930b096a 100644
--- a/pages/queries/pages/alan-harness-tracking-field-all.page-query.ts
+++ b/pages/queries/pages/alan-harness-tracking-field-all.page-query.ts
@@ -5,5 +5,5 @@ export const alanHarnessTrackingFieldAll = {
   pageTypeSlug: "page-query",
   slug: "alan-harness-tracking-field-all",
   asksOfSlug: "tracking-field",
-  keys: ["slug", "key", "defined-on-slug", "type"],
+  keys: ["slug", "propertySlug", "pageTypeSlug"],
 } as const satisfies PageQuery
diff --git a/pages/queries/pages/page-property-definition-all.page-query.ts b/pages/queries/pages/page-property-definition-all.page-query.ts
index c00f83f093642a3e512e702899ed71836e82f144..cd9631e81eae88aa804af1e152ebe25da5ed4c2c 100644
--- a/pages/queries/pages/page-property-definition-all.page-query.ts
+++ b/pages/queries/pages/page-property-definition-all.page-query.ts
@@ -5,5 +5,5 @@ export const pagePropertyDefinitionAll = {
   pageTypeSlug: "page-query",
   slug: "page-property-definition-all",
   asksOfSlug: "page-property",
-  keys: ["slug", "key", "defined-on-slug", "type"],
+  keys: ["slug", "propertySlug", "pageTypeSlug"],
 } as const satisfies PageQuery
diff --git a/pages/queries/pages/persona-all.page-query.ts b/pages/queries/pages/persona-all.page-query.ts
index 58b3a450217930f08ed832c83eced29dc5610ec7..5479e50c0dc989a717d48e193bcc81083d549f61 100644
--- a/pages/queries/pages/persona-all.page-query.ts
+++ b/pages/queries/pages/persona-all.page-query.ts
@@ -8,7 +8,6 @@ export const personaAll = {
   keys: [
     "id",
     "slug",
-    "seq",
     "title",
     "cover",
     "valueSlug",
diff --git a/pages/queries/pages/relation-property-definitions.page-query.ts b/pages/queries/pages/relation-property-definitions.page-query.ts
index a83bc3da6c2997e74246c2522f55ec0736777c84..853b23518fbefb56d45b020d9cc21ab527db3879 100644
--- a/pages/queries/pages/relation-property-definitions.page-query.ts
+++ b/pages/queries/pages/relation-property-definitions.page-query.ts
@@ -5,6 +5,6 @@ export const relationPropertyDefinitions = {
   pageTypeSlug: "page-query",
   slug: "relation-property-definitions",
   asksOfSlug: "page-property",
-  narrows: [{ key: "type", comparison: "is", values: ["relation-id"] }],
-  keys: ["defined-on-slug", "key"],
+  narrows: [{ key: "pageTypeSlug", comparison: "is", values: ["relation-property"] }],
+  keys: ["slug", "propertySlug"],
 } as const satisfies PageQuery
