writerOwesReading: false
diff --git a/personas/aria/aria.persona.uncommitted.ts b/personas/aria/aria.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..71c42643c40b88efc3c36cfc96cb1342e0d80ecd
--- /dev/null
+++ b/personas/aria/aria.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const ariaPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-31T16:21:59.370Z",
+} as const
diff --git a/personas/atlas/atlas.persona.uncommitted.ts b/personas/atlas/atlas.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..06f070b3cf6aef01dd6da78f7d76c8ef06af1621
--- /dev/null
+++ b/personas/atlas/atlas.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const atlasPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-31T21:48:08.761Z",
+} as const
diff --git a/personas/awen/awen.persona.uncommitted.ts b/personas/awen/awen.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..d64c5a2e0d8f7671c239ca7ed9ffbf8422808049
--- /dev/null
+++ b/personas/awen/awen.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const awenPersonaUncommitted = {
+  "lastMessagedAt": "2026-08-19T23:14:31.620Z",
+} as const
diff --git a/personas/ceri/ceri.persona.uncommitted.ts b/personas/ceri/ceri.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..15b410ca73eb27c8ab1a5f6dc66e99cdce8f7b0b
--- /dev/null
+++ b/personas/ceri/ceri.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const ceriPersonaUncommitted = {
+  "lastMessagedAt": "2026-08-13T20:16:11.888Z",
+} as const
diff --git a/personas/claude/claude.persona.uncommitted.ts b/personas/claude/claude.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..e73ed586c9e677bb29246bf72c76e58aadd9ba88
--- /dev/null
+++ b/personas/claude/claude.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const claudePersonaUncommitted = {
+  "lastMessagedAt": "2026-08-25T12:26:50.332Z",
+} as const
diff --git a/personas/echo/echo.persona.uncommitted.ts b/personas/echo/echo.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..b26a66ec135364024c9993d6d78e775a144df76f
--- /dev/null
+++ b/personas/echo/echo.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const echoPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-25T10:56:15.401Z",
+} as const
diff --git a/personas/elaine/elaine.persona.uncommitted.ts b/personas/elaine/elaine.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..4f42e3647c77f7faabdca0260cb8b3ab5babdc14
--- /dev/null
+++ b/personas/elaine/elaine.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const elainePersonaUncommitted = {
+  "lastMessagedAt": "2026-06-27T01:41:39.215Z",
+} as const
diff --git a/personas/elin/elin.persona.uncommitted.ts b/personas/elin/elin.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..de0398e368f7fa524401cb9e249ee1fbf3a23e53
--- /dev/null
+++ b/personas/elin/elin.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const elinPersonaUncommitted = {
+  "lastMessagedAt": "2026-08-01T17:14:37.422Z",
+} as const
diff --git a/personas/eppie/eppie.persona.uncommitted.ts b/personas/eppie/eppie.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..eba48ffa721642156eab40dffaa44283ae211cd2
--- /dev/null
+++ b/personas/eppie/eppie.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const eppiePersonaUncommitted = {
+  "lastMessagedAt": "2026-06-30T11:48:49.049Z",
+} as const
diff --git a/personas/erin/erin.persona.uncommitted.ts b/personas/erin/erin.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..dc5022f7b9e847f2d0082499070536fe98575132
--- /dev/null
+++ b/personas/erin/erin.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const erinPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-02T16:55:28.670Z",
+} as const
diff --git a/personas/grace/grace.persona.uncommitted.ts b/personas/grace/grace.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..07ff08cb816dc774165ccc1489470128c038941a
--- /dev/null
+++ b/personas/grace/grace.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const gracePersonaUncommitted = {
+  "lastMessagedAt": "2026-07-01T19:06:56.191Z",
+} as const
diff --git a/personas/ione/ione.persona.uncommitted.ts b/personas/ione/ione.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..a8dd23538e32acec9fd43a28a978154b946a9f7a
--- /dev/null
+++ b/personas/ione/ione.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const ionePersonaUncommitted = {
+  "lastMessagedAt": "2026-07-18T05:03:49.242Z",
+} as const
diff --git a/personas/lali/lali.persona.uncommitted.ts b/personas/lali/lali.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..00fb8ae2b1bfd60391a15abb7769575d181dd4ef
--- /dev/null
+++ b/personas/lali/lali.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const laliPersonaUncommitted = {
+  "lastMessagedAt": "2026-08-11T12:29:35.270Z",
+} as const
diff --git a/personas/mari/mari.persona.uncommitted.ts b/personas/mari/mari.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..e02670d91f31b62b68d74453a2065dabbc430e93
--- /dev/null
+++ b/personas/mari/mari.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const mariPersonaUncommitted = {
+  "lastMessagedAt": "2026-08-29T03:35:38.911Z",
+} as const
diff --git a/personas/natalie/natalie.persona.uncommitted.ts b/personas/natalie/natalie.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..b9402f79975dca19bd5dad45bbabd4b37b59547a
--- /dev/null
+++ b/personas/natalie/natalie.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const nataliePersonaUncommitted = {
+  "lastMessagedAt": "2026-07-25T13:55:22.447Z",
+} as const
diff --git a/personas/nova/nova.persona.uncommitted.ts b/personas/nova/nova.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..170ec2bc08e12e7e0cd842f103053b718570fc03
--- /dev/null
+++ b/personas/nova/nova.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const novaPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-18T07:09:01.201Z",
+} as const
diff --git a/personas/rhia/rhia.persona.uncommitted.ts b/personas/rhia/rhia.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..f697d4a1aaa52e6016fa79ab1701ff3fff84487c
--- /dev/null
+++ b/personas/rhia/rhia.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const rhiaPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-24T17:56:50.240Z",
+} as const
diff --git a/personas/ruby/ruby.persona.uncommitted.ts b/personas/ruby/ruby.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..924f5b4d27e029e046b54b76c1c2da8b4059c082
--- /dev/null
+++ b/personas/ruby/ruby.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const rubyPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-24T12:39:50.970Z",
+} as const
diff --git a/personas/selah/selah.persona.uncommitted.ts b/personas/selah/selah.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..73bec6edd859c97cd882703ecb2323e59563c64b
--- /dev/null
+++ b/personas/selah/selah.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const selahPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-24T20:50:14.590Z",
+} as const
diff --git a/personas/shaestrel/shaestrel.persona.uncommitted.ts b/personas/shaestrel/shaestrel.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..6c18e393f6a695395712ffe365e222e2d51cbd7e
--- /dev/null
+++ b/personas/shaestrel/shaestrel.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const shaestrelPersonaUncommitted = {
+  "lastMessagedAt": "2026-07-25T12:48:20.868Z",
+} as const
diff --git a/personas/sophia/sophia.persona.uncommitted.ts b/personas/sophia/sophia.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..c8801724f30c4542c207c52e32e133b4680fa689
--- /dev/null
+++ b/personas/sophia/sophia.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const sophiaPersonaUncommitted = {
+  "lastMessagedAt": "2026-08-30T20:09:25.968Z",
+} as const
diff --git a/personas/talia/talia.persona.uncommitted.ts b/personas/talia/talia.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..72b7bb4bd40818fe485c95140c68f5fb03acec35
--- /dev/null
+++ b/personas/talia/talia.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const taliaPersonaUncommitted = {
+  "lastMessagedAt": "2026-06-28T04:06:00.366Z",
+} as const
diff --git a/personas/vera/vera.persona.uncommitted.ts b/personas/vera/vera.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..5a6483aeae171e1e056c6cb09e22f1150aae88c2
--- /dev/null
+++ b/personas/vera/vera.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const veraPersonaUncommitted = {
+  "lastMessagedAt": "2026-08-28T01:50:59.381Z",
+} as const
diff --git a/personas/zadi/zadi.persona.uncommitted.ts b/personas/zadi/zadi.persona.uncommitted.ts
new file mode 100644
index 0000000000000000000000000000000000000000..0b110ed4622c5f950f462749449987e899ca2a4b
--- /dev/null
+++ b/personas/zadi/zadi.persona.uncommitted.ts
@@ -0,0 +1,3 @@
+export const zadiPersonaUncommitted = {
+  "lastMessagedAt": "2026-06-28T18:38:19.631Z",
+} as const
