diff --git a/command-system/commands/command.page-type.ts b/command-system/commands/command.page-type.ts
index ab3066d47f522e4e1d1f559d2a00a6ab50dcd0e0..614df2840f7ea85c36542569c74a23ff1fef15a7 100644
--- a/command-system/commands/command.page-type.ts
+++ b/command-system/commands/command.page-type.ts
@@ -43,6 +43,7 @@ export const command = {
     "command/compose-notices",
     "command/deploy",
     "command/dev-server",
+    "command/domain",
     "command/domain-tree",
     "command/drive",
     "command/edit",
