diff --git a/domains/domains.workspace-package.ts b/domains/domains.workspace-package.ts
index c24804b4889e1e63677bc73370670899ae0dcc74..51f8c4e3d322a80695221bf3605d6d8116ec5c42 100644
--- a/domains/domains.workspace-package.ts
+++ b/domains/domains.workspace-package.ts
@@ -20,7 +20,6 @@ export const domains = {
     "domain/domain-purpose",
     "domain/domain-champions",
     "domain/domain-parent",
-    "command/domain",
     "module/domain-reading",
   ],
   invariants: [
