diff --git a/readout-system/stoplight/stoplight.domain.ts b/readout-system/stoplight/stoplight.domain.ts
index e7754409f774877b4034139a47d911185a11ea3f..c1d9e9df0a98888a338b5092f9d27cacb7d91414 100644
--- a/readout-system/stoplight/stoplight.domain.ts
+++ b/readout-system/stoplight/stoplight.domain.ts
@@ -8,7 +8,11 @@ export const stoplight = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A stoplight has five colors, worst to best: black, red, yellow, green, blue.",
+      statement: "A stoplight has five colors.",
+    },
+    {
+      invariantKind: "departure",
+      statement: "The colors from worst to best are black and red and yellow and green and blue.",
     },
     {
       invariantKind: "departure",
