readersOweReading: false
diff --git a/pages/service/page-calling/page-calling.module.ts b/pages/service/page-calling/page-calling.module.ts
index d5545a98d683363d212caa6680b8662777c5f482..d2e0aeecbc134816f7bd873de979e25213b088c6 100644
--- a/pages/service/page-calling/page-calling.module.ts
+++ b/pages/service/page-calling/page-calling.module.ts
@@ -35,7 +35,7 @@ export const pageCalling = {
     },
     {
       invariantKind: "departure",
-      statement: "A caller may put its own fetcher in place of the global one.",
+      statement: "A caller may put its own fetcher in place of the global fetcher.",
     },
     {
       invariantKind: "departure",
