readersOweReading: false
diff --git a/git/porcelain-status/porcelain-status.module.ts b/git/porcelain-status/porcelain-status.module.ts
index 105cccb04c2c229728dc21941974e15718768a79..af9bb3d57cae019a4153d7852e9d98463c8cc7d2 100644
--- a/git/porcelain-status/porcelain-status.module.ts
+++ b/git/porcelain-status/porcelain-status.module.ts
@@ -17,7 +17,7 @@ export const porcelainStatus = {
     },
     {
       invariantKind: "departure",
-      statement: "A record is two status columns, a space, and the path.",
+      statement: "A record is two status columns and a space and the path.",
     },
     {
       invariantKind: "departure",
