diff --git a/collections/collection.page-type.ts b/collections/collection.page-type.ts
index 7bcc832c90e0a2842d943bce0a6b3ae53c60b3ab..6326cef3d3ce6d0fbe4bacfc6ceae653580ccf4f 100644
--- a/collections/collection.page-type.ts
+++ b/collections/collection.page-type.ts
@@ -2,7 +2,6 @@ import type { Page } from "@akasha/pages/page"
 import type { PageType } from "@akasha/pages/page-type"
 import type { CollectionAuthor } from "./properties/collection-author.text-property.ts"
 import type { CollectionCompletedAt } from "./properties/collection-completed-at.instant-property.ts"
-import type { CollectionDescription } from "./properties/collection-description.text-property.ts"
 import type { CollectionPublishedAt } from "./properties/collection-published-at.one-of-property.ts"
 import type { CollectionTags } from "./properties/collection-tags.text-property.ts"
 import type { CollectionTypeSlug } from "./properties/collection-type-slug.relation-property.ts"
@@ -21,7 +20,6 @@ import type { UnitWords } from "./properties/unit-words.number-property.ts"
 export type Collection = Page & {
   author?: CollectionAuthor
   completedAt?: CollectionCompletedAt
-  description?: CollectionDescription
   following?: Following
   ownLength?: OwnLength
   ownProgress?: OwnProgress
@@ -80,7 +78,6 @@ export const collection = {
     "relation-property/unit-slug",
     "select-property/status",
     "text-property/collection-author",
-    "text-property/collection-description",
     "text-property/collection-tags",
     "workspace-package/great-courses",
     "workspace-package/royal-road",
