readersOweReading: false
diff --git a/checks/code-checks/pages/no-refused-syntax/syntax-rules/no-non-null-assertion/no-non-null-assertion.syntax-rule.ts b/checks/code-checks/pages/no-refused-syntax/syntax-rules/no-non-null-assertion/no-non-null-assertion.syntax-rule.ts
index a057452ef77f4c73b0e69b47cdc8231a4debd93f..7a7c05de28540e772841e9bf2a3085c7c08fbeaa 100644
--- a/checks/code-checks/pages/no-refused-syntax/syntax-rules/no-non-null-assertion/no-non-null-assertion.syntax-rule.ts
+++ b/checks/code-checks/pages/no-refused-syntax/syntax-rules/no-non-null-assertion/no-non-null-assertion.syntax-rule.ts
@@ -19,7 +19,7 @@ export const noNonNullAssertion = {
     },
     {
       invariantKind: "departure",
-      statement: "One reached through an optional chain is refused as a bare one is.",
+      statement: "An operator reached through an optional chain is refused as a bare operator is.",
     },
     {
       invariantKind: "departure",
