readersOweReading: false
diff --git a/pages/core/view/page-query-times/page-query-times.module.ts b/pages/core/view/page-query-times/page-query-times.module.ts
index e7741633d762f3b14c6675d1a0683ae1275a13a7..eedd2ef8204cf4ff0290ae77efb3afe285239b49 100644
--- a/pages/core/view/page-query-times/page-query-times.module.ts
+++ b/pages/core/view/page-query-times/page-query-times.module.ts
@@ -10,11 +10,11 @@ export const pageQueryTimes = {
     { invariantKind: "departure", statement: "Now is the moment the query is answered." },
     {
       invariantKind: "departure",
-      statement: "An eso day is counted from six in the morning, New York.",
+      statement: "An eso day is counted from six in the morning New York time.",
     },
     {
       invariantKind: "departure",
-      statement: "The next eso day is the one after the eso day the query is answered in.",
+      statement: "The next eso day is the eso day after the eso day the query is answered in.",
     },
     {
       invariantKind: "departure",
