writerOwesReading: false
diff --git a/domains/initiatives/pages/ryn-standard-agent-english.initiative.ts b/domains/initiatives/pages/ryn-standard-agent-english.initiative.ts
index 871eca0d191b0a8807e09549d3f8aebe0f3dd42e..7f9c789335a289e635899db40b8f01c38f0f748d 100644
--- a/domains/initiatives/pages/ryn-standard-agent-english.initiative.ts
+++ b/domains/initiatives/pages/ryn-standard-agent-english.initiative.ts
@@ -10,6 +10,12 @@ export const rynStandardAgentEnglish = {
     {
       statement: "The load a construction puts on a reading agent is measured.",
     },
+    {
+      statement: "Standard Agent English defines what terms are allowed.",
+    },
+    {
+      statement: "Standard Agent English defines what constructions are allowed.",
+    },
   ],
   constraints: [
     "The load a construction puts on a reading agent is measured.",
