readersOweReading: false
diff --git a/machines/clusters/cluster.page-type.ts b/machines/clusters/cluster.page-type.ts
index 880cfdde6fe1fa85e5d1a1285c114b9846ab5c34..1b829a84db19cf1d302ecb7b56bd99ec34301181 100644
--- a/machines/clusters/cluster.page-type.ts
+++ b/machines/clusters/cluster.page-type.ts
@@ -28,7 +28,7 @@ export const cluster = {
     {
       invariantKind: "gap",
       statement:
-        "The work a person did on one of those machines under Windows, that person can still do.",
+        "The work a person did on a cluster machine under Windows is work that person can still do.",
     },
     {
       invariantKind: "gap",
