readersOweReading: false
diff --git a/seat-system/supervising/supervisor-spawn-settings/supervisor-spawn-settings.module.ts b/seat-system/supervising/supervisor-spawn-settings/supervisor-spawn-settings.module.ts
index bfc63d01b5ab60dfa6633e5cb5d1fa5138eb6653..88bd0b90d0dd984b8512ebdb63520db75a0f596a 100644
--- a/seat-system/supervising/supervisor-spawn-settings/supervisor-spawn-settings.module.ts
+++ b/seat-system/supervising/supervisor-spawn-settings/supervisor-spawn-settings.module.ts
@@ -14,7 +14,7 @@ export const supervisorSpawnSettings = {
     {
       invariantKind: "departure",
       statement:
-        "The agent settings module is imported from beside this one and its function called.",
+        "The agent settings module is imported from beside this module and its function called.",
     },
     {
       invariantKind: "departure",
