diff --git a/alan/tracking/to-dos/to-do.page-type.ts b/alan/tracking/to-dos/to-do.page-type.ts
index 52ea53867e4b8157a852f093c4cf12729d068abb..3381038ede61a927e56d9fefe20d04b0a6d258d3 100644
--- a/alan/tracking/to-dos/to-do.page-type.ts
+++ b/alan/tracking/to-dos/to-do.page-type.ts
@@ -87,7 +87,7 @@ export const toDo = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A recurring to-do earns once a day, however many times that to-do is completed.",
+      statement: "A recurring to-do earns once a day however many times that to-do is completed.",
     },
     {
       invariantKind: "departure",
