readersOweReading: false
diff --git a/service-system/workstation-services/properties/needs-secrets.boolean-property.ts b/service-system/workstation-services/properties/needs-secrets.boolean-property.ts
index fdcaae386bd3e2f632b6112c3298d3bc76731020..87969c2e41a24180adf0d6e36894c42c17be2807 100644
--- a/service-system/workstation-services/properties/needs-secrets.boolean-property.ts
+++ b/service-system/workstation-services/properties/needs-secrets.boolean-property.ts
@@ -16,7 +16,7 @@ export const needsSecrets = {
     },
     {
       invariantKind: "departure",
-      statement: "A service stating nothing is handed none.",
+      statement: "A service stating nothing is handed no secret.",
     },
   ],
 } as const satisfies BooleanProperty
