readersOweReading: false
diff --git a/git/oid.domain.ts b/git/oid.domain.ts
index dcee5c6e19e5f8f00164e42d728b281a5cff0987..6122e759cc9858ff6b3604efd7b3e7411b5c4029 100644
--- a/git/oid.domain.ts
+++ b/git/oid.domain.ts
@@ -8,7 +8,7 @@ export const oid = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "Every name for a git object id is `oid`, or a scoped form of `oid`.",
+      statement: "Every name for a git object id is `oid` or a scoped form of `oid`.",
     },
   ],
 } as const satisfies Domain
