readersOweReading: false
diff --git a/temper/temper-items-rules-matcher/inventory-rule-matcher-allocators/inventory-rule-matcher-allocators.module.ts b/temper/temper-items-rules-matcher/inventory-rule-matcher-allocators/inventory-rule-matcher-allocators.module.ts
index 9706f1c49a23314eaa6248cbc805c7244f9c6c62..741c04bae9ccf19c5270c37b3d464f56c1cba8d8 100644
--- a/temper/temper-items-rules-matcher/inventory-rule-matcher-allocators/inventory-rule-matcher-allocators.module.ts
+++ b/temper/temper-items-rules-matcher/inventory-rule-matcher-allocators/inventory-rule-matcher-allocators.module.ts
@@ -13,7 +13,7 @@ export const inventoryRuleMatcherAllocators = {
     },
     {
       invariantKind: "departure",
-      statement: "A rule takes no more of an item than the rule asked for.",
+      statement: "A rule takes no larger amount of an item than the rule asked for.",
     },
   ],
 } as const satisfies Module
