readersOweReading: false
diff --git a/temper/temper-events-addon/events-addon-global/events-addon-global.module.ts b/temper/temper-events-addon/events-addon-global/events-addon-global.module.ts
index 4dda457cf057ddf5aaf71a6abf2ebb6fb3a49721..735c5e793efbf2aa3277ec01a22b103f8f8c98dd 100644
--- a/temper/temper-events-addon/events-addon-global/events-addon-global.module.ts
+++ b/temper/temper-events-addon/events-addon-global/events-addon-global.module.ts
@@ -10,7 +10,7 @@ export const eventsAddonGlobal = {
     {
       invariantKind: "departure",
       statement:
-        "Another add-on reaches this one through the global rather than through an import.",
+        "Another add-on reaches this add-on through the global rather than through an import.",
     },
     {
       invariantKind: "departure",
