readersOweReading: false
diff --git a/temper/temper-lib-sets/lib-sets-kbf-drop-filters/lib-sets-kbf-drop-filters.module.ts b/temper/temper-lib-sets/lib-sets-kbf-drop-filters/lib-sets-kbf-drop-filters.module.ts
index 2c27b5530cb128343337777c5017f1558e6b969e..2447ec63126b74e3818931000005a3609d0a09b9 100644
--- a/temper/temper-lib-sets/lib-sets-kbf-drop-filters/lib-sets-kbf-drop-filters.module.ts
+++ b/temper/temper-lib-sets/lib-sets-kbf-drop-filters/lib-sets-kbf-drop-filters.module.ts
@@ -9,7 +9,7 @@ export const libSetsKbfDropFilters = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "A zone id of zero or less names a special zone rather than a game zone.",
+      statement: "A zone id at zero or below names a special zone rather than a game zone.",
     },
   ],
 } as const satisfies Module
