readersOweReading: false
diff --git a/idle-system/idle-persona-cards/idle-persona-card.page-type.ts b/idle-system/idle-persona-cards/idle-persona-card.page-type.ts
index 7cf1ba2b204d349785db108d9d9986a1ff2ad3a9..f993db293aacda1d693033a6d6168c3b1106b5eb 100644
--- a/idle-system/idle-persona-cards/idle-persona-card.page-type.ts
+++ b/idle-system/idle-persona-cards/idle-persona-card.page-type.ts
@@ -77,7 +77,7 @@ export const idlePersonaCard = {
     },
     {
       invariantKind: "departure",
-      statement: "A card is slugged by the card it is and then the player who holds it.",
+      statement: "A card is slugged by the card slug and then the player holding that card.",
     },
     {
       invariantKind: "departure",
