readersOweReading: false
diff --git a/checks/code-checks/pages/folder-matches-a-shape/folder-shapes/a-workstation-service-with-its-parts/a-workstation-service-with-its-parts.folder-shape.ts b/checks/code-checks/pages/folder-matches-a-shape/folder-shapes/a-workstation-service-with-its-parts/a-workstation-service-with-its-parts.folder-shape.ts
index 9adc347f16fbdaaae98f6bd42db8d31d3f49b96e..52c37e9bdbf4cc0267ada8e5dd3d746a4ffe9a2d 100644
--- a/checks/code-checks/pages/folder-matches-a-shape/folder-shapes/a-workstation-service-with-its-parts/a-workstation-service-with-its-parts.folder-shape.ts
+++ b/checks/code-checks/pages/folder-matches-a-shape/folder-shapes/a-workstation-service-with-its-parts/a-workstation-service-with-its-parts.folder-shape.ts
@@ -15,7 +15,7 @@ export const aWorkstationServiceWithItsParts = {
     },
     {
       invariantKind: "departure",
-      statement: "It holds one page.",
+      statement: "The folder holds one page.",
     },
     {
       invariantKind: "departure",
