readersOweReading: false
diff --git a/temper/temper-addon-data/addon-data-writes-companion-mappings/addon-data-writes-companion-mappings.module.ts b/temper/temper-addon-data/addon-data-writes-companion-mappings/addon-data-writes-companion-mappings.module.ts
index c4b83886ce02abf3021f56d736997d50baba3367..0d4030172bed71cd683e9b1806bde5be047b9232 100644
--- a/temper/temper-addon-data/addon-data-writes-companion-mappings/addon-data-writes-companion-mappings.module.ts
+++ b/temper/temper-addon-data/addon-data-writes-companion-mappings/addon-data-writes-companion-mappings.module.ts
@@ -15,7 +15,7 @@ export const addonDataWritesCompanionMappings = {
     },
     {
       invariantKind: "departure",
-      statement: "A section states its writes rather than performing those writes.",
+      statement: "A section states its writes rather than performing a write.",
     },
   ],
 } as const satisfies Module
