diff --git a/temper/temper-character/character-builds/character-build.page-type.ts b/temper/temper-character/character-builds/character-build.page-type.ts
index abf96c587ba4c7dc828ccfcf7b9ebf30de38d2cd..73a3c906c8d007753cda8948150c016f7d016325 100644
--- a/temper/temper-character/character-builds/character-build.page-type.ts
+++ b/temper/temper-character/character-builds/character-build.page-type.ts
@@ -38,7 +38,11 @@ export const characterBuild = {
     },
     {
       invariantKind: "departure",
-      statement: "One character holds many builds, and one build is shared across characters.",
+      statement: "One character holds many builds.",
+    },
+    {
+      invariantKind: "departure",
+      statement: "One build is shared across characters.",
     },
     {
       invariantKind: "departure",
