readersOweReading: false
diff --git a/temper/temper-characters-addon/characters-hireling-mail-subscriber/characters-hireling-mail-subscriber.module.ts b/temper/temper-characters-addon/characters-hireling-mail-subscriber/characters-hireling-mail-subscriber.module.ts
index f258f511bed450d3d13f008052a8a6767d95335b..0a35884fac1f71007d4b988bdc49a3fbb3a1e4e7 100644
--- a/temper/temper-characters-addon/characters-hireling-mail-subscriber/characters-hireling-mail-subscriber.module.ts
+++ b/temper/temper-characters-addon/characters-hireling-mail-subscriber/characters-hireling-mail-subscriber.module.ts
@@ -9,7 +9,7 @@ export const charactersHirelingMailSubscriber = {
   invariants: [
     {
       invariantKind: "absence",
-      statement: "A loot of nothing or less is ignored.",
+      statement: "A loot of nothing or a negative amount is ignored.",
     },
   ],
 } as const satisfies Module
