readersOweReading: false
diff --git a/service-system/workstation-services/properties/port.number-property.ts b/service-system/workstation-services/properties/port.number-property.ts
index 8108b85ca86e8222b2da9ff72e2e1b81cc207c1b..86525e2371cfd66d717e254018a6e3075e6fbe87 100644
--- a/service-system/workstation-services/properties/port.number-property.ts
+++ b/service-system/workstation-services/properties/port.number-property.ts
@@ -16,7 +16,7 @@ export const port = {
     },
     {
       invariantKind: "departure",
-      statement: "A service stating none listens nowhere.",
+      statement: "A service stating no port listens nowhere.",
     },
   ],
 } as const satisfies NumberProperty
