readersOweReading: false
diff --git a/temper/temper-player-completion/completion-generic-checker-progress/completion-generic-checker-progress.module.ts b/temper/temper-player-completion/completion-generic-checker-progress/completion-generic-checker-progress.module.ts
index 5af91b5b82e4ddd6d12c5658903f940d3e2b43b4..612a46177000fc9323a81b1f60ea5d7323f66932 100644
--- a/temper/temper-player-completion/completion-generic-checker-progress/completion-generic-checker-progress.module.ts
+++ b/temper/temper-player-completion/completion-generic-checker-progress/completion-generic-checker-progress.module.ts
@@ -14,7 +14,8 @@ export const completionGenericCheckerProgress = {
     },
     {
       invariantKind: "departure",
-      statement: "A leaf stating its own numbers is taken over counting that leaf as one of one.",
+      statement:
+        "A leaf stating its own numbers is taken over counting that leaf as a single whole.",
     },
     {
       invariantKind: "departure",
