readersOweReading: false
diff --git a/idle-system/idle-saves/idle-save.page-type.ts b/idle-system/idle-saves/idle-save.page-type.ts
index c4d1317a2605e2c2c4eafd230fd060ee44a38668..978ef2b70686b1c58bc28dc9ba85a50e97cc7145 100644
--- a/idle-system/idle-saves/idle-save.page-type.ts
+++ b/idle-system/idle-saves/idle-save.page-type.ts
@@ -13,7 +13,7 @@ export const idleSave = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A player has one save and no more.",
+      statement: "A player has one save and no second save.",
     },
     {
       invariantKind: "departure",
