readersOweReading: false
diff --git a/temper/temper-lib-data-encode/temper-lib-data-encode.eso-addon.ts b/temper/temper-lib-data-encode/temper-lib-data-encode.eso-addon.ts
index 485f50a37d7ca11bfa664e3531eb153eea7abd53..ea7ca7c286986ae57afb8be50873e3d79406945f 100644
--- a/temper/temper-lib-data-encode/temper-lib-data-encode.eso-addon.ts
+++ b/temper/temper-lib-data-encode/temper-lib-data-encode.eso-addon.ts
@@ -37,7 +37,7 @@ export const temperLibDataEncode = {
     {
       invariantKind: "departure",
       statement:
-        "A value seen three times or more is put in a dictionary and referred to by index.",
+        "A value seen at least three times is put in a dictionary and referred to by index.",
     },
     {
       invariantKind: "departure",
