readersOweReading: false
diff --git a/file-system/data-place/data-place.module.ts b/file-system/data-place/data-place.module.ts
index ab1c465263b1b431c818980f6ada04f9c270b8a4..c8fd61be54980f1ed07a8779cf3147400f10654a 100644
--- a/file-system/data-place/data-place.module.ts
+++ b/file-system/data-place/data-place.module.ts
@@ -58,7 +58,7 @@ export const dataPlace = {
     },
     {
       invariantKind: "absence",
-      statement: "None of those places is known here.",
+      statement: "Those places are not known here.",
     },
   ],
 } as const satisfies Module
