readersOweReading: false
diff --git a/story/games/game.page-type.ts b/story/games/game.page-type.ts
index 113e1d01a228e959f35fd3fec105b36efd5ba44f..0ca26c4d8fcd04683d2944e9a332d95eafcfa10d 100644
--- a/story/games/game.page-type.ts
+++ b/story/games/game.page-type.ts
@@ -176,7 +176,8 @@ export const game = {
     },
     {
       invariantKind: "departure",
-      statement: "A game character is one a player runs rather than one the game runs.",
+      statement:
+        "A game character is a character a player runs rather than a character the game runs.",
     },
     {
       invariantKind: "departure",
