readersOweReading: false
diff --git a/infrastructure/analysis-complexity/operator-classification/operator-classification.module.ts b/infrastructure/analysis-complexity/operator-classification/operator-classification.module.ts
index 5a04e192b8d66f4b515626348033f4c30806f521..07e0dab5421f4d3aca6236197bc2865a9dc8a256 100644
--- a/infrastructure/analysis-complexity/operator-classification/operator-classification.module.ts
+++ b/infrastructure/analysis-complexity/operator-classification/operator-classification.module.ts
@@ -9,7 +9,7 @@ export const operatorClassification = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A token is an operator, an operand, or uncounted.",
+      statement: "A token is an operator or an operand or uncounted.",
     },
     {
       invariantKind: "departure",
