readersOweReading: false
diff --git a/temper/temper-addon-data/addon-data-writes-inventory/addon-data-writes-inventory.module.ts b/temper/temper-addon-data/addon-data-writes-inventory/addon-data-writes-inventory.module.ts
index 61a00356c538772287d073888c83f2c5ba163c53..2b55dbe29952e8afd8fa382242dbaf264656600c 100644
--- a/temper/temper-addon-data/addon-data-writes-inventory/addon-data-writes-inventory.module.ts
+++ b/temper/temper-addon-data/addon-data-writes-inventory/addon-data-writes-inventory.module.ts
@@ -13,7 +13,7 @@ export const addonDataWritesInventory = {
     },
     {
       invariantKind: "departure",
-      statement: "A section states its writes rather than performing those writes.",
+      statement: "A section states its writes rather than performing a write.",
     },
   ],
 } as const satisfies Module
