readersOweReading: false
diff --git a/mobile-commands/mobile-cut-record/mobile-cut-record.command.ts b/mobile-commands/mobile-cut-record/mobile-cut-record.command.ts
index 9b1fb24fe75ff9322e64dca5b6065735b3f80394..50c8fe538242f07e1bbfe86216902d1126b9b584 100644
--- a/mobile-commands/mobile-cut-record/mobile-cut-record.command.ts
+++ b/mobile-commands/mobile-cut-record/mobile-cut-record.command.ts
@@ -32,7 +32,7 @@ export const mobileCutRecord = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A build number is a whole number of at least one.",
+      statement: "A build number is a whole number no lower than 1.",
     },
     {
       invariantKind: "departure",
