readersOweReading: false
diff --git a/temper/temper-build-deploy-checks/addon-sandbox-load/addon-sandbox-load.module.ts b/temper/temper-build-deploy-checks/addon-sandbox-load/addon-sandbox-load.module.ts
index a05a09f31e334610506c2fd111884acf50659ffc..3de89f3ee2217078cbccc11c0a66524ee61b1535 100644
--- a/temper/temper-build-deploy-checks/addon-sandbox-load/addon-sandbox-load.module.ts
+++ b/temper/temper-build-deploy-checks/addon-sandbox-load/addon-sandbox-load.module.ts
@@ -13,7 +13,7 @@ export const addonSandboxLoad = {
     },
     {
       invariantKind: "constraint",
-      statement: "A traceback longer than ten lines is shortened and says how many went.",
+      statement: "A traceback longer than ten lines is shortened and says how many lines went.",
     },
     {
       invariantKind: "constraint",
