readersOweReading: false
diff --git a/workspace-paths/tsconfig-references/tsconfig-references.module.ts b/workspace-paths/tsconfig-references/tsconfig-references.module.ts
index ba6e66c35e9828aa8e993cf781cd0094c54bfd6a..7e8096757c7aff80b0a2a3d67e83965343c496b9 100644
--- a/workspace-paths/tsconfig-references/tsconfig-references.module.ts
+++ b/workspace-paths/tsconfig-references/tsconfig-references.module.ts
@@ -15,7 +15,7 @@ export const tsconfigReferences = {
     {
       invariantKind: "departure",
       statement:
-        "The tsconfig files read are the two at the repository root and the one in each workspace folder.",
+        "The tsconfig files read are the two at the repository root and the file in each workspace folder.",
     },
     {
       invariantKind: "departure",
