diff --git a/alan/web-capacitor/alan-web-capacitor.router-app.ts b/alan/web-capacitor/alan-web-capacitor.router-app.ts
index be4022acd2bac65aac8a9fde761bd6fbf8a541f4..13841688d3a0a025d94223b96369d1e441baf99e 100644
--- a/alan/web-capacitor/alan-web-capacitor.router-app.ts
+++ b/alan/web-capacitor/alan-web-capacitor.router-app.ts
@@ -11,6 +11,7 @@ export const alanWebCapacitor = {
   serverEntry: "tsx",
   routerConfig: "ts",
   compileConfig: "json",
+  viteConfig: "ts",
   partSlugs: ["route/alan-web-capacitor-home", "stylesheet/alan-web-capacitor-look"],
   invariants: [
     {
