readersOweReading: false
diff --git a/temper/temper-keybinder-addon/keybinder-share/keybinder-share.module.ts b/temper/temper-keybinder-addon/keybinder-share/keybinder-share.module.ts
index e0b88247a790442d48b64ee272bd5a93c7eb487e..27082c4e39558217d49a13f3a7b5a4d483fa41de 100644
--- a/temper/temper-keybinder-addon/keybinder-share/keybinder-share.module.ts
+++ b/temper/temper-keybinder-addon/keybinder-share/keybinder-share.module.ts
@@ -18,8 +18,7 @@ export const keybinderShare = {
     },
     {
       invariantKind: "departure",
-      statement:
-        "The toggle clears the shown actions where every one of those actions is already shared.",
+      statement: "The toggle clears the shown actions where those actions are already shared.",
     },
     {
       invariantKind: "departure",
