readersOweReading: false
diff --git a/temper/temper-lib-sets/lib-sets-tip-hooks/lib-sets-tip-hooks.module.ts b/temper/temper-lib-sets/lib-sets-tip-hooks/lib-sets-tip-hooks.module.ts
index 51bcba1b680dc02979a05c10ee23b57f1fa21f34..1ea3faa1c7451dbb6e8fc13e53aa174ac5270360 100644
--- a/temper/temper-lib-sets/lib-sets-tip-hooks/lib-sets-tip-hooks.module.ts
+++ b/temper/temper-lib-sets/lib-sets-tip-hooks/lib-sets-tip-hooks.module.ts
@@ -15,7 +15,7 @@ export const libSetsTipHooks = {
     {
       invariantKind: "constraint",
       statement:
-        "An existing handler on another add-on's control keeps being called before this one.",
+        "An existing handler on another add-on's control keeps being called before this handler.",
     },
     {
       invariantKind: "constraint",
