readersOweReading: false
diff --git a/seat-system/supervising/supervisor-idle-rule/supervisor-idle-rule.module.ts b/seat-system/supervising/supervisor-idle-rule/supervisor-idle-rule.module.ts
index c1a8825d9de4546f36cb4cc4524aa33b14cd9a1f..7189063eb579b56c583dc0ea954173316c1a1d13 100644
--- a/seat-system/supervising/supervisor-idle-rule/supervisor-idle-rule.module.ts
+++ b/seat-system/supervising/supervisor-idle-rule/supervisor-idle-rule.module.ts
@@ -13,7 +13,7 @@ export const supervisorIdleRule = {
     },
     {
       invariantKind: "departure",
-      statement: "The safe answer for cmdlines is that none of those cmdlines are ignored.",
+      statement: "The safe answer for cmdlines is that no cmdline is ignored.",
     },
     {
       invariantKind: "departure",
