readersOweReading: false
diff --git a/domains/parent/domain-parent.domain.ts b/domains/parent/domain-parent.domain.ts
index 2814f6f8bb69356f4b49bdf5c773ef93fe32e1ef..0ca3b467dbdaf530ca0d615e0405ccfd401ff22a 100644
--- a/domains/parent/domain-parent.domain.ts
+++ b/domains/parent/domain-parent.domain.ts
@@ -9,7 +9,7 @@ export const domainParent = {
     {
       invariantKind: "departure",
       statement:
-        "A domain needed only to read this one is required reading rather than its parent.",
+        "A domain needed only to read this domain is required reading rather than its parent.",
     },
     {
       invariantKind: "departure",
