diff --git a/checks/cluster-checks/modules/functional-type/functional-type.module.ts b/checks/cluster-checks/modules/functional-type/functional-type.module.ts
index eee53851e8fefdcb1a49e1dd505726672034cdaf..e6b39c606f21f28151b5a5604f857280638c3c29 100644
--- a/checks/cluster-checks/modules/functional-type/functional-type.module.ts
+++ b/checks/cluster-checks/modules/functional-type/functional-type.module.ts
@@ -30,11 +30,15 @@ export const functionalType = {
     },
     {
       invariantKind: "departure",
-      statement: "A library type is one that any package may depend on.",
+      statement: "A library type is a type any package may depend on.",
     },
     {
       invariantKind: "departure",
-      statement: "A type carries a rank, and a package may not depend on a rank above its own.",
+      statement: "A type carries a rank.",
+    },
+    {
+      invariantKind: "departure",
+      statement: "A package may not depend on a rank above its own.",
     },
     {
       invariantKind: "departure",
