readersOweReading: false
diff --git a/alan/fitness/workout-schedules/workout-schedule.page-type.ts b/alan/fitness/workout-schedules/workout-schedule.page-type.ts
index 0134203a773e248d36146d31b342e87aec3b9d1e..a4fcf64b08b63b12a4ae7c3f742a79e18d2b5bd6 100644
--- a/alan/fitness/workout-schedules/workout-schedule.page-type.ts
+++ b/alan/fitness/workout-schedules/workout-schedule.page-type.ts
@@ -33,7 +33,7 @@ export const workoutSchedule = {
   invariants: [
     {
       invariantKind: "upkeep",
-      statement: "Exactly one schedule is the active one.",
+      statement: "Exactly one schedule is active.",
     },
   ],
 } as const satisfies PageType
