diff --git a/pages/address/page-address.module.ts b/pages/address/page-address.module.ts
index de1f677f6276590fbe3008b8cb5e39e2de6c31f1..5eedcb08fa05d1c541c0b12d6896f2cceb132ff4 100644
--- a/pages/address/page-address.module.ts
+++ b/pages/address/page-address.module.ts
@@ -14,7 +14,11 @@ export const pageAddress = {
     },
     {
       invariantKind: "departure",
-      statement: "A slug carries no slash, because a slug is lower kebab case.",
+      statement: "A slug carries no slash.",
+    },
+    {
+      invariantKind: "departure",
+      statement: "A slug is lower kebab case.",
     },
     {
       invariantKind: "departure",
