readersOweReading: false
diff --git a/temper/temper-items-addon/inventory-config-file/inventory-config-file.lua-module.ts b/temper/temper-items-addon/inventory-config-file/inventory-config-file.lua-module.ts
index 0ed6b3bb94d07b0cca20726911c8057683816b4f..089c3a0da6ed65a8fdda271c5306814950941887 100644
--- a/temper/temper-items-addon/inventory-config-file/inventory-config-file.lua-module.ts
+++ b/temper/temper-items-addon/inventory-config-file/inventory-config-file.lua-module.ts
@@ -16,7 +16,7 @@ export const inventoryConfigFile = {
     {
       invariantKind: "departure",
       statement:
-        "The game loads the file before the compiled add-on rather than compiling the two as one.",
+        "The game loads the file before the compiled add-on rather than compiling the two together.",
     },
   ],
 } as const satisfies LuaModule
