writerOwesReading: false
diff --git a/infrastructure/alerts/properties/alert-description.file-property.ts b/infrastructure/alerts/properties/alert-description.file-property.ts
index e0cf14beac4f07a13062fbd71fc23d0bdadb3774..ff9ed145a8f02c08c05931e239d1903e5ebb0471 100644
--- a/infrastructure/alerts/properties/alert-description.file-property.ts
+++ b/infrastructure/alerts/properties/alert-description.file-property.ts
@@ -6,7 +6,7 @@ export const alertDescription = {
   id: "01a06755-0778-7275-8076-4aec6002693b",
   pageTypeSlug: "file-property",
   slug: "alert-description",
-  propertySlug: "description",
+  propertySlug: "runbook",
   definition: "what to check once an alert is raised",
   invariants: [
     {
