diff --git a/changes/kinds/change-kind.page-type.ts b/changes/kinds/change-kind.page-type.ts
index 68ecc772541b26976c51142d67d850b7976fb1dc..96abc5b66d82ca1755561248316d00d50d172174 100644
--- a/changes/kinds/change-kind.page-type.ts
+++ b/changes/kinds/change-kind.page-type.ts
@@ -1,11 +1,11 @@
 import type { Domain } from "../../domains/domain.page-type.ts"
 import type { PageType } from "../../pages/types/page-type.page-type.ts"
 import type { RunsChecks } from "./properties/runs-checks.boolean-property.ts"
-import type { RunsWarrants } from "./properties/writer-owes-reading.boolean-property.ts"
+import type { WriterOwesReading } from "./properties/writer-owes-reading.boolean-property.ts"
 
 export type ChangeKind = Domain & {
   runsChecks: RunsChecks
-  writerOwesReading: RunsWarrants
+  writerOwesReading: WriterOwesReading
 }
 
 export const changeKind = {
diff --git a/changes/kinds/properties/writer-owes-reading.boolean-property.ts b/changes/kinds/properties/writer-owes-reading.boolean-property.ts
index fc18b9082cf82f04100cbe30943f01f6b931ecac..6dd611d12b07f21f8d89eeaeae8a6f1b737f6953 100644
--- a/changes/kinds/properties/writer-owes-reading.boolean-property.ts
+++ b/changes/kinds/properties/writer-owes-reading.boolean-property.ts
@@ -1,6 +1,6 @@
 import type { BooleanProperty } from "../../../pages/boolean-properties/boolean-property.page-type.ts"
 
-export type RunsWarrants = boolean
+export type WriterOwesReading = boolean
 
 export const writerOwesReading = {
   id: "01a05e19-7ffb-7e64-a651-4f26f37b2e7b",
