diff --git a/akasha.domain.ts b/akasha.domain.ts
index 8e81be6228c943757f3a7236aad0ee58c48e411c..2d3cb13de6b1a777041567d7a36dfb81cfa54466 100644
--- a/akasha.domain.ts
+++ b/akasha.domain.ts
@@ -70,20 +70,6 @@ export const akasha = {
     "workspace-package/mobile-commands",
     "workspace/akasha-workspace",
   ],
-  invariants: [
-    {
-      invariantKind: "departure",
-      statement: "The repository is a database.",
-    },
-    {
-      invariantKind: "departure",
-      statement: "This tree is its master replica.",
-    },
-    {
-      invariantKind: "departure",
-      statement: "What is missing was lost rather than not yet made.",
-    },
-  ],
   directives: [
     {
       directiveKind: "rule",
