writerOwesReading: false
readersOweReading: false
diff --git a/command-system/commands/page-tree/page-tree.command.test.ts b/command-system/commands/page-tree/page-tree.command.test.ts
index e95aeef7dcb1c14a2b682fc91366a078e6d59c2a..190df52bfb5008cfa221b9943ac3197e1f5754e8 100644
--- a/command-system/commands/page-tree/page-tree.command.test.ts
+++ b/command-system/commands/page-tree/page-tree.command.test.ts
@@ -23,7 +23,7 @@ const TYPES: readonly Valued[] = [
       extendsSlug: ["page-type/page"],
       properties: [
         { pagePropertySlug: "title", required: true, many: false },
-        { pagePropertySlug: "note", required: false, many: true, max: 3 },
+        { pagePropertySlug: "note", required: false, many: true, maxCount: 3 },
       ],
     },
   },
