writerOwesReading: false
readersOweReading: false
diff --git a/places/locations/properties/scheduled-start.instant-property.ts b/places/locations/properties/scheduled-start.instant-property.ts
deleted file mode 100644
index 32b927d2b209790653e4dce5571d9712f0f9966c..0000000000000000000000000000000000000000
--- a/places/locations/properties/scheduled-start.instant-property.ts
+++ /dev/null
@@ -1,11 +0,0 @@
-import type { InstantProperty } from "@akasha/pages/instant-property"
-
-export type ScheduledStart = string
-
-export const scheduledStart = {
-  id: "01a06583-acfb-73ae-b66c-ba755708a687",
-  pageTypeSlug: "instant-property",
-  slug: "scheduled-start",
-  propertySlug: "scheduled-start",
-  definition: "when the person is due at the place",
-} as const satisfies InstantProperty
