diff --git a/code-system/components/component.page-type.ts b/code-system/components/component.page-type.ts
index 45bf173aefbc5fc02c24eb4d1b5c09a935d6b7a8..a09886de1fc5d1e7ce45b8d19eb575db7190ff11 100644
--- a/code-system/components/component.page-type.ts
+++ b/code-system/components/component.page-type.ts
@@ -1,10 +1,13 @@
 import type { Domain } from "@akasha/domains/domain"
 import type { PageType } from "@akasha/pages-system/page-type"
+import type { ComponentCode } from "./properties/component-code.file-property.ts"
+import type { ComponentTest } from "./properties/component-test.file-property.ts"
+import type { ComponentTestFixtures } from "./properties/component-test-fixtures.file-property.ts"
 
 export type Component = Domain & {
-  code: "tsx"
-  test?: "tsx"
-  testFixtures?: "tsx"
+  code: ComponentCode
+  test?: ComponentTest
+  testFixtures?: ComponentTestFixtures
 }
 
 export const component = {
