diff --git a/code-system/routes/route.page-type.ts b/code-system/routes/route.page-type.ts
index 1203e838bef037d1252a96e0a27f60adce46e311..22afe9b010f2fec13cd09c5c7e29a6d360c6d20a 100644
--- a/code-system/routes/route.page-type.ts
+++ b/code-system/routes/route.page-type.ts
@@ -1,10 +1,13 @@
 import type { Domain } from "@akasha/domains/domain"
 import type { PageType } from "@akasha/pages-system/page-type"
+import type { RouteCode } from "./properties/route-code.file-property.ts"
+import type { RouteTest } from "./properties/route-test.file-property.ts"
+import type { UrlPath } from "./properties/url-path.text-property.ts"
 
 export type Route = Domain & {
-  code: "ts" | "tsx"
-  test?: "ts" | "tsx"
-  urlPath?: string
+  code: RouteCode
+  test?: RouteTest
+  urlPath?: UrlPath
 }
 
 export const route = {
