writerOwesReading: false
readersOweReading: false
diff --git a/pages/properties/description.text-property.ts b/pages/properties/description.text-property.ts
index 3795363c86e81f4fa31fd677366e1222263d045a..5ad2da05e8480db0117c615fbabab642409a49c6 100644
--- a/pages/properties/description.text-property.ts
+++ b/pages/properties/description.text-property.ts
@@ -8,6 +8,6 @@ export const description = {
   slug: "description",
   propertySlug: "description",
   definition: "what a thing is, said for a reader",
-  max: 2000,
+  maxLength: 2000,
   nameFormatSlug: null,
 } as const satisfies TextProperty
diff --git a/pages/properties/slug.text-property.ts b/pages/properties/slug.text-property.ts
index e972eb85a8c9fe6ae2dd5cc3b90a6fae2351aa07..acce29580698342ac26a3947b4cef3b38d0fe75b 100644
--- a/pages/properties/slug.text-property.ts
+++ b/pages/properties/slug.text-property.ts
@@ -8,7 +8,7 @@ export const slug = {
   slug: "slug",
   propertySlug: "slug",
   definition: "the name a page is reached by",
-  max: 100,
+  maxLength: 100,
   nameFormatSlug: "name-format/lower-kebab-case",
   unique: "page-type",
   invariants: [
