writerOwesReading: false
readersOweReading: false
diff --git a/pages/indexes/property-carrying/property-carrying.module.code.ts b/pages/indexes/property-carrying/property-carrying.module.code.ts
index 31918d32cc69664824f2e163a997f4d80cf21d89..f058025bfd6ea4d329ef1c9ccd4a04f9a1c2326d 100644
--- a/pages/indexes/property-carrying/property-carrying.module.code.ts
+++ b/pages/indexes/property-carrying/property-carrying.module.code.ts
@@ -1,5 +1,5 @@
 import { basename, dirname } from "node:path"
-import { partedIn } from "@akasha/pages/page-file-name"
+import { partedIn, sectionedIn } from "@akasha/pages/page-file-name"
 import type { Value } from "@akasha/pages/page-value"
 import {
   everyOfType,
