diff --git a/service-system/secrets/pages/realtime-secrets-database-url.secret.ts b/service-system/secrets/pages/realtime-secrets-database-url.secret.ts
new file mode 100644
index 0000000000000000000000000000000000000000..43824a2331921681394e31262bac1ece4d327ea3
--- /dev/null
+++ b/service-system/secrets/pages/realtime-secrets-database-url.secret.ts
@@ -0,0 +1,8 @@
+import type { Secret } from "../secret.page-type.ts"
+
+export const realtimeSecretsDatabaseUrl = {
+  id: "01a07699-44e5-72b2-b43e-d748516645bf",
+  pageTypeSlug: "secret",
+  slug: "realtime-secrets-database-url",
+  placements: [{ resourceName: "realtime-secrets", resourceKey: "DATABASE_URL" }],
+} as const satisfies Secret
