diff --git a/infrastructure/networks/headscale/headscale.manifest.ts b/infrastructure/networks/headscale/headscale.manifest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..5306809af85982580b1e92dfd271368b66ca545f
--- /dev/null
+++ b/infrastructure/networks/headscale/headscale.manifest.ts
@@ -0,0 +1,10 @@
+import type { Manifest } from "@akasha/k8s-types/manifest"
+
+export const headscale = {
+  id: "01a0738c-9157-7c4c-906c-8ff65da0976d",
+  pageTypeSlug: "manifest",
+  slug: "headscale",
+  definition:
+    "the workload, service and policies running the server that admits machines to the private network",
+  code: "ts",
+} as const satisfies Manifest
diff --git a/infrastructure/networks/network.domain.ts b/infrastructure/networks/network.domain.ts
index 60c729de1e432ed6803e3cae0f3452af756c9563..a234e0d22fc597f50867496ff6ee11cdf8623f87 100644
--- a/infrastructure/networks/network.domain.ts
+++ b/infrastructure/networks/network.domain.ts
@@ -10,6 +10,7 @@ export const network = {
     "domain/certificate",
     "manifest/cloudflared",
     "manifest/ddns-headscale",
+    "manifest/headscale",
     "workspace-package/auth-proxy",
   ],
   invariants: [
