diff --git a/infrastructure/telemetry/postgres-exporter/postgres-exporter.manifest.ts b/infrastructure/telemetry/postgres-exporter/postgres-exporter.manifest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..60111aad4e2574358da015eb77e25a1be6a2e383
--- /dev/null
+++ b/infrastructure/telemetry/postgres-exporter/postgres-exporter.manifest.ts
@@ -0,0 +1,9 @@
+import type { Manifest } from "@akasha/k8s-types/manifest"
+
+export const postgresExporter = {
+  id: "01a07396-d8bf-7514-af72-2499f6246428",
+  pageTypeSlug: "manifest",
+  slug: "postgres-exporter",
+  definition: "the database exporter's queries, its deployment and the way in to it",
+  code: "ts",
+} as const satisfies Manifest
diff --git a/infrastructure/telemetry/telemetry.domain.ts b/infrastructure/telemetry/telemetry.domain.ts
index 8426df71a6085ff0fb53a43c9047555edf8d5b31..54a280c0dafa069d1f9052c2c2d9f610f535a0cf 100644
--- a/infrastructure/telemetry/telemetry.domain.ts
+++ b/infrastructure/telemetry/telemetry.domain.ts
@@ -12,5 +12,6 @@ export const telemetry = {
     "manifest/kube-state-metrics",
     "manifest/node-exporter-daemonset",
     "manifest/pgbouncer-exporter",
+    "manifest/postgres-exporter",
   ],
 } as const satisfies Domain
