diff --git a/pages/indexes/index-reading/index-reading.module.code.ts b/pages/indexes/index-reading/index-reading.module.code.ts
index 58f21e09db9a2844a409ab83ec1512d414878e51..06ff4dd2f8d58ab8cfb866709571eea8c0961a24 100644
--- a/pages/indexes/index-reading/index-reading.module.code.ts
+++ b/pages/indexes/index-reading/index-reading.module.code.ts
@@ -1,5 +1,4 @@
 import { join } from "node:path"
-import { typed } from "@akasha/code-system/code-typing"
 import { addressIn } from "@akasha/pages-system/page-address"
 import { partedIn } from "@akasha/pages-system/page-file-name"
 import type { Value } from "@akasha/pages-system/page-value"
@@ -10,7 +9,6 @@ import { indexRelation } from "../index/relation/index-relation.index.ts"
 import { indexSchema } from "../index/schema/index-schema.index.ts"
 import { indexValue } from "../index/value/index-value.index.ts"
 import type { Reading } from "../index-shape/index-shape.module.code.ts"
-import { staleFor } from "../index-stamp/index-stamp.module.code.ts"
 import {
   beneath,
   INDEX_AT,
