diff --git a/infrastructure/telemetry/pgbouncer-exporter/pgbouncer-exporter.manifest.code.ts b/infrastructure/telemetry/pgbouncer-exporter/pgbouncer-exporter.manifest.code.ts
new file mode 100644
index 0000000000000000000000000000000000000000..acfd4239a5bd262d0d6c6736b17dc3db68c1e1a7
--- /dev/null
+++ b/infrastructure/telemetry/pgbouncer-exporter/pgbouncer-exporter.manifest.code.ts
@@ -0,0 +1,11 @@
+import {
+  pgbouncerExporterDeploymentYaml,
+  pgbouncerExporterServiceYaml,
+} from "@akasha/cluster-manifests/pgbouncer-exporter-manifests"
+
+export default function synth(): readonly { readonly name: string; readonly yaml: string }[] {
+  return [
+    { name: "pgbouncer-exporter-deployment", yaml: pgbouncerExporterDeploymentYaml() },
+    { name: "pgbouncer-exporter-service", yaml: pgbouncerExporterServiceYaml() },
+  ]
+}
diff --git a/service-system/cluster-services/pages/pgbouncer-exporter/pgbouncer-exporter.cluster-service.code.attachment.ts b/service-system/cluster-services/pages/pgbouncer-exporter/pgbouncer-exporter.cluster-service.code.attachment.ts
deleted file mode 100644
index acfd4239a5bd262d0d6c6736b17dc3db68c1e1a7..0000000000000000000000000000000000000000
--- a/service-system/cluster-services/pages/pgbouncer-exporter/pgbouncer-exporter.cluster-service.code.attachment.ts
+++ /dev/null
@@ -1,11 +0,0 @@
-import {
-  pgbouncerExporterDeploymentYaml,
-  pgbouncerExporterServiceYaml,
-} from "@akasha/cluster-manifests/pgbouncer-exporter-manifests"
-
-export default function synth(): readonly { readonly name: string; readonly yaml: string }[] {
-  return [
-    { name: "pgbouncer-exporter-deployment", yaml: pgbouncerExporterDeploymentYaml() },
-    { name: "pgbouncer-exporter-service", yaml: pgbouncerExporterServiceYaml() },
-  ]
-}
diff --git a/service-system/cluster-services/pages/pgbouncer-exporter/pgbouncer-exporter.cluster-service.ts b/service-system/cluster-services/pages/pgbouncer-exporter/pgbouncer-exporter.cluster-service.ts
index 6c570862a95dbcb8fa0be42f2783db250f376359..43bd3c1ebdddb0c3c290e911e58930e09c00f759 100644
--- a/service-system/cluster-services/pages/pgbouncer-exporter/pgbouncer-exporter.cluster-service.ts
+++ b/service-system/cluster-services/pages/pgbouncer-exporter/pgbouncer-exporter.cluster-service.ts
@@ -11,6 +11,5 @@ export const pgbouncerExporter = {
   image: "prometheuscommunity/pgbouncer-exporter:v0.12.0",
   replicas: 1,
   containerPort: 9127,
-  manifestCode:
-    "service-system/cluster-services/pages/pgbouncer-exporter/pgbouncer-exporter.cluster-service.code.attachment.ts",
+  manifestSlug: "pgbouncer-exporter",
 } as const satisfies ClusterService
