diff --git a/temper/temper-web/manifests/temper-web-manifests.manifest.ts b/temper/temper-web/manifests/temper-web-manifests.manifest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..fe87a4c2b5e4168d167edf166c3962e44e0a221d
--- /dev/null
+++ b/temper/temper-web/manifests/temper-web-manifests.manifest.ts
@@ -0,0 +1,9 @@
+import type { Manifest } from "@akasha/k8s-types/manifest"
+
+export const temperWebManifests = {
+  id: "01a07384-ba71-7e7e-83f4-b7564fa65954",
+  pageTypeSlug: "manifest",
+  slug: "temper-web-manifests",
+  definition: "the deployment and service serving the parts of Temper that run in a browser",
+  code: "ts",
+} as const satisfies Manifest
diff --git a/temper/temper-web/temper-web.router-app.ts b/temper/temper-web/temper-web.router-app.ts
index 62457435a26df9292d0224858906930f319e2b99..4a9f211c2de0d250ae85343c12d1602b77b16636 100644
--- a/temper/temper-web/temper-web.router-app.ts
+++ b/temper/temper-web/temper-web.router-app.ts
@@ -12,6 +12,7 @@ export const temperWeb = {
   routerConfig: "ts",
   compileConfig: "json",
   partSlugs: [
+    "manifest/temper-web-manifests",
     "stylesheet/temper-web-look",
     "module/account-achievements-panel-card",
     "module/account-collectibles-panel-card",
