diff --git a/service-system/cluster-services/pages/page-store/page-store.cluster-service.ts b/service-system/cluster-services/pages/page-store/page-store.cluster-service.ts
index 0d572f593b8b19db6606e94714cf4aeedfc37f54..6fa08466942bff0f68bbc842914c474369f0320c 100644
--- a/service-system/cluster-services/pages/page-store/page-store.cluster-service.ts
+++ b/service-system/cluster-services/pages/page-store/page-store.cluster-service.ts
@@ -11,8 +11,7 @@ export const pageStore = {
   image: "alpine/socat:1.8.0.3",
   replicas: 1,
   containerPort: 8787,
-  manifestCode:
-    "service-system/cluster-services/pages/page-store/page-store.cluster-service.code.attachment.ts",
+  manifestSlug: "page-store",
   invariants: [
     {
       invariantKind: "departure",
