diff --git a/service-system/secrets/pages/realtime-secrets-api-jwt-secret.secret.ts b/service-system/secrets/pages/realtime-secrets-api-jwt-secret.secret.ts
new file mode 100644
index 0000000000000000000000000000000000000000..affc60120d5773a3c1a70219946e0b8e336741f4
--- /dev/null
+++ b/service-system/secrets/pages/realtime-secrets-api-jwt-secret.secret.ts
@@ -0,0 +1,8 @@
+import type { Secret } from "../secret.page-type.ts"
+
+export const realtimeSecretsApiJwtSecret = {
+  id: "01a07699-9fc1-7678-b91d-ef6a1a22b047",
+  pageTypeSlug: "secret",
+  slug: "realtime-secrets-api-jwt-secret",
+  placements: [{ resourceName: "realtime-secrets", resourceKey: "API_JWT_SECRET" }],
+} as const satisfies Secret
diff --git a/service-system/secrets/pages/realtime-secrets-database-url.secret.ts b/service-system/secrets/pages/realtime-secrets-database-url.secret.ts
new file mode 100644
index 0000000000000000000000000000000000000000..43824a2331921681394e31262bac1ece4d327ea3
--- /dev/null
+++ b/service-system/secrets/pages/realtime-secrets-database-url.secret.ts
@@ -0,0 +1,8 @@
+import type { Secret } from "../secret.page-type.ts"
+
+export const realtimeSecretsDatabaseUrl = {
+  id: "01a07699-44e5-72b2-b43e-d748516645bf",
+  pageTypeSlug: "secret",
+  slug: "realtime-secrets-database-url",
+  placements: [{ resourceName: "realtime-secrets", resourceKey: "DATABASE_URL" }],
+} as const satisfies Secret
diff --git a/service-system/secrets/pages/realtime-secrets-db-enc-key.secret.ts b/service-system/secrets/pages/realtime-secrets-db-enc-key.secret.ts
new file mode 100644
index 0000000000000000000000000000000000000000..853f5294775b7077bceaf02735306ee8f51d7e7e
--- /dev/null
+++ b/service-system/secrets/pages/realtime-secrets-db-enc-key.secret.ts
@@ -0,0 +1,8 @@
+import type { Secret } from "../secret.page-type.ts"
+
+export const realtimeSecretsDbEncKey = {
+  id: "01a07699-6219-7063-a25d-8050d0f6f272",
+  pageTypeSlug: "secret",
+  slug: "realtime-secrets-db-enc-key",
+  placements: [{ resourceName: "realtime-secrets", resourceKey: "DB_ENC_KEY" }],
+} as const satisfies Secret
diff --git a/service-system/secrets/pages/realtime-secrets-metrics-jwt-secret.secret.ts b/service-system/secrets/pages/realtime-secrets-metrics-jwt-secret.secret.ts
new file mode 100644
index 0000000000000000000000000000000000000000..1b6aacdc34a41020ac28fb4343abe102b23af28a
--- /dev/null
+++ b/service-system/secrets/pages/realtime-secrets-metrics-jwt-secret.secret.ts
@@ -0,0 +1,8 @@
+import type { Secret } from "../secret.page-type.ts"
+
+export const realtimeSecretsMetricsJwtSecret = {
+  id: "01a07699-dde0-7222-a8c2-61b2342cc09d",
+  pageTypeSlug: "secret",
+  slug: "realtime-secrets-metrics-jwt-secret",
+  placements: [{ resourceName: "realtime-secrets", resourceKey: "METRICS_JWT_SECRET" }],
+} as const satisfies Secret
diff --git a/service-system/secrets/pages/realtime-secrets-secret-key-base.secret.ts b/service-system/secrets/pages/realtime-secrets-secret-key-base.secret.ts
new file mode 100644
index 0000000000000000000000000000000000000000..f887ff5a7c85dd6df149de0ecc1f8f5d51a14191
--- /dev/null
+++ b/service-system/secrets/pages/realtime-secrets-secret-key-base.secret.ts
@@ -0,0 +1,8 @@
+import type { Secret } from "../secret.page-type.ts"
+
+export const realtimeSecretsSecretKeyBase = {
+  id: "01a07699-bf5d-7bed-b050-0703e319d6c1",
+  pageTypeSlug: "secret",
+  slug: "realtime-secrets-secret-key-base",
+  placements: [{ resourceName: "realtime-secrets", resourceKey: "SECRET_KEY_BASE" }],
+} as const satisfies Secret
diff --git a/service-system/secrets/pages/supabase-realtime-admin-secrets-supabase-realtime-admin-password.secret.ts b/service-system/secrets/pages/supabase-realtime-admin-secrets-supabase-realtime-admin-password.secret.ts
new file mode 100644
index 0000000000000000000000000000000000000000..bbc68aec679e0c47f197cc6a9d6b3084081cab9f
--- /dev/null
+++ b/service-system/secrets/pages/supabase-realtime-admin-secrets-supabase-realtime-admin-password.secret.ts
@@ -0,0 +1,13 @@
+import type { Secret } from "../secret.page-type.ts"
+
+export const supabaseRealtimeAdminSecretsSupabaseRealtimeAdminPassword = {
+  id: "01a07699-fcc8-7fa7-b472-7fc397441e81",
+  pageTypeSlug: "secret",
+  slug: "supabase-realtime-admin-secrets-supabase-realtime-admin-password",
+  placements: [
+    {
+      resourceName: "supabase-realtime-admin-secrets",
+      resourceKey: "SUPABASE_REALTIME_ADMIN_PASSWORD",
+    },
+  ],
+} as const satisfies Secret
