diff --git a/infrastructure/telemetry/pgbouncer-exporter/pgbouncer-exporter.manifest.ts b/infrastructure/telemetry/pgbouncer-exporter/pgbouncer-exporter.manifest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..a8bf1f77704f5460e25ca26850b2eb0be6f12262
--- /dev/null
+++ b/infrastructure/telemetry/pgbouncer-exporter/pgbouncer-exporter.manifest.ts
@@ -0,0 +1,9 @@
+import type { Manifest } from "@akasha/k8s-types/manifest"
+
+export const pgbouncerExporter = {
+  id: "01a07395-071f-7a7b-959d-53aefec229ea",
+  pageTypeSlug: "manifest",
+  slug: "pgbouncer-exporter",
+  definition: "the connection pool exporter's deployment and the way in to it",
+  code: "ts",
+} as const satisfies Manifest
diff --git a/infrastructure/telemetry/telemetry.domain.ts b/infrastructure/telemetry/telemetry.domain.ts
index ead491189d7f5259d6d938c99a81addefcaa2fea..8426df71a6085ff0fb53a43c9047555edf8d5b31 100644
--- a/infrastructure/telemetry/telemetry.domain.ts
+++ b/infrastructure/telemetry/telemetry.domain.ts
@@ -11,5 +11,6 @@ export const telemetry = {
     "manifest/dcgm-exporter-daemonset",
     "manifest/kube-state-metrics",
     "manifest/node-exporter-daemonset",
+    "manifest/pgbouncer-exporter",
   ],
 } as const satisfies Domain
