diff --git a/infrastructure/telemetry/grafana/grafana.manifest.ts b/infrastructure/telemetry/grafana/grafana.manifest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..6de193ea31dc7306dd2611fe6d2d87c829b61ddd
--- /dev/null
+++ b/infrastructure/telemetry/grafana/grafana.manifest.ts
@@ -0,0 +1,10 @@
+import type { Manifest } from "@akasha/k8s-types/manifest"
+
+export const grafana = {
+  id: "01a07386-5069-75df-b299-2b356475469a",
+  pageTypeSlug: "manifest",
+  slug: "grafana",
+  definition:
+    "the chart server's deployment, its namespace, its configuration and the way in to it",
+  code: "ts",
+} as const satisfies Manifest
