diff --git a/audhdalan/audhdalan-web/audhdalan-web.router-app.ts b/audhdalan/audhdalan-web/audhdalan-web.router-app.ts
index a2c7f275f4c84ee54e56dc06992d185c07a8ca53..78a8d804548092c3f6ba338fde47f8265371a2c3 100644
--- a/audhdalan/audhdalan-web/audhdalan-web.router-app.ts
+++ b/audhdalan/audhdalan-web/audhdalan-web.router-app.ts
@@ -12,6 +12,7 @@ export const audhdalanWeb = {
   routerConfig: "ts",
   compileConfig: "json",
   partSlugs: [
+    "manifest/audhdalan-web-manifests",
     "stylesheet/audhdalan-web-look",
     "module/deck-page-content",
     "module/deck-slides",
diff --git a/audhdalan/audhdalan-web/manifests/audhdalan-web-manifests.manifest.ts b/audhdalan/audhdalan-web/manifests/audhdalan-web-manifests.manifest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..a640e56a3e991f658160f56172554e66b4d7516d
--- /dev/null
+++ b/audhdalan/audhdalan-web/manifests/audhdalan-web-manifests.manifest.ts
@@ -0,0 +1,9 @@
+import type { Manifest } from "@akasha/k8s-types/manifest"
+
+export const audhdalanWebManifests = {
+  id: "01a07388-768e-7f43-89ee-b66e2a21d046",
+  pageTypeSlug: "manifest",
+  slug: "audhdalan-web-manifests",
+  definition: "the deployment and service serving the site Alan's neurodiversity writing is on",
+  code: "ts",
+} as const satisfies Manifest
