diff --git a/infrastructure/telemetry/postgres-exporter/postgres-exporter.manifest.ts b/infrastructure/telemetry/postgres-exporter/postgres-exporter.manifest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..60111aad4e2574358da015eb77e25a1be6a2e383
--- /dev/null
+++ b/infrastructure/telemetry/postgres-exporter/postgres-exporter.manifest.ts
@@ -0,0 +1,9 @@
+import type { Manifest } from "@akasha/k8s-types/manifest"
+
+export const postgresExporter = {
+  id: "01a07396-d8bf-7514-af72-2499f6246428",
+  pageTypeSlug: "manifest",
+  slug: "postgres-exporter",
+  definition: "the database exporter's queries, its deployment and the way in to it",
+  code: "ts",
+} as const satisfies Manifest
