diff --git a/infrastructure/seaweedfs/maintenance/seaweedfs-maintenance.manifest.ts b/infrastructure/seaweedfs/maintenance/seaweedfs-maintenance.manifest.ts
new file mode 100644
index 0000000000000000000000000000000000000000..7b99f663a6a8619a8fdeedf7f0b0b2c363633a77
--- /dev/null
+++ b/infrastructure/seaweedfs/maintenance/seaweedfs-maintenance.manifest.ts
@@ -0,0 +1,10 @@
+import type { Manifest } from "@akasha/k8s-types/manifest"
+
+export const seaweedfsMaintenance = {
+  id: "01a07380-3b60-7ede-8cda-e469a40f6113",
+  pageTypeSlug: "manifest",
+  slug: "seaweedfs-maintenance",
+  definition:
+    "the namespace and the cron job rebalancing and compacting the volumes files are held in",
+  code: "ts",
+} as const satisfies Manifest
