runsWarrants: false
diff --git a/pages/indexes/identity/index-identity.index.ts b/pages/indexes/identity/index-identity.index.ts
index 2efacae860bfe7efea018223a8698ba17c5a695d..e2242127508ef2e2a746e5e9dfe8e9b727f6bc1f 100644
--- a/pages/indexes/identity/index-identity.index.ts
+++ b/pages/indexes/identity/index-identity.index.ts
@@ -16,7 +16,7 @@ export const indexIdentity = {
     {
       invariantKind: "departure",
       statement:
-        "The scope is what an identifier is unique within, a page type or a page's parent.",
+        "The scope is the reach an identifier is unique within, a page type or a page's parent.",
     },
     {
       invariantKind: "departure",
