runsWarrants: false
diff --git a/code-system/body-text/body-text.module.ts b/code-system/body-text/body-text.module.ts
index e06605716593e2ba9cb0edd61023c227f7b37f8b..7a97ecd0e4a9d619703c32412530eed032ffd022 100644
--- a/code-system/body-text/body-text.module.ts
+++ b/code-system/body-text/body-text.module.ts
@@ -14,7 +14,7 @@ export const bodyText = {
     },
     {
       invariantKind: "departure",
-      statement: "A path with nothing at it says nothing rather than empty text.",
+      statement: "A path with nothing at that path says nothing rather than empty text.",
     },
     {
       invariantKind: "constraint",
