runsWarrants: false
diff --git a/alan/fitness/session-planning/next-set/next-set.module.ts b/alan/fitness/session-planning/next-set/next-set.module.ts
index 868822b9f218d53ebaf757dfd525e17127104805..335d27d65bce7059c6c0dcf0cb14033f11af7952 100644
--- a/alan/fitness/session-planning/next-set/next-set.module.ts
+++ b/alan/fitness/session-planning/next-set/next-set.module.ts
@@ -30,7 +30,8 @@ export const nextSet = {
     },
     {
       invariantKind: "departure",
-      statement: "The next set says why the movement before it ended, unless it simply finished.",
+      statement:
+        "The next set says why the movement before that set ended, unless that movement simply finished.",
     },
     {
       invariantKind: "departure",
