runsWarrants: false
diff --git a/temper/temper-addon-data/addon-data-writes-sets/addon-data-writes-sets.module.ts b/temper/temper-addon-data/addon-data-writes-sets/addon-data-writes-sets.module.ts
index 040e9585d735c92720ade9c3bc230a88139554cf..117f1db88c83946c7ccacfacc113fbc5433305fd 100644
--- a/temper/temper-addon-data/addon-data-writes-sets/addon-data-writes-sets.module.ts
+++ b/temper/temper-addon-data/addon-data-writes-sets/addon-data-writes-sets.module.ts
@@ -13,7 +13,7 @@ export const addonDataWritesSets = {
     },
     {
       invariantKind: "departure",
-      statement: "A section states its writes rather than performing them.",
+      statement: "A section states its writes rather than performing those writes.",
     },
   ],
 } as const satisfies Module
