runsWarrants: false
diff --git a/google/email/commands/email-command-reading/email-command-reading.module.ts b/google/email/commands/email-command-reading/email-command-reading.module.ts
index 75de5ec1faab1b8a6f9828284fa2585a55ebaad3..2abcadc66c70c0b616ebcc9c42d918d960dc6559 100644
--- a/google/email/commands/email-command-reading/email-command-reading.module.ts
+++ b/google/email/commands/email-command-reading/email-command-reading.module.ts
@@ -64,7 +64,7 @@ export const emailCommandReading = {
     },
     {
       invariantKind: "departure",
-      statement: "What a command answers is JSON, one report line for each line of that JSON.",
+      statement: "A command answers JSON, one report line for each line of that JSON.",
     },
     {
       invariantKind: "departure",
