runsWarrants: false
diff --git a/sms-core/commands/sms-send/sms-send.command.ts b/sms-core/commands/sms-send/sms-send.command.ts
index 110801b0a8912f4f63676576a4e4d0b756363b27..0500404e08a85874591ce0eabcb921cfdab59da8 100644
--- a/sms-core/commands/sms-send/sms-send.command.ts
+++ b/sms-core/commands/sms-send/sms-send.command.ts
@@ -40,7 +40,7 @@ export const smsSend = {
     },
     {
       invariantKind: "departure",
-      statement: "The number sent from is the toll-free one unless another is said.",
+      statement: "The number sent from is the toll-free one unless another number is said.",
     },
     {
       invariantKind: "departure",
