runsWarrants: false
diff --git a/temper/temper-lib-zone/zone-names-es-03/zone-names-es-03.module.ts b/temper/temper-lib-zone/zone-names-es-03/zone-names-es-03.module.ts
index 317bfa25b526788cbaf8d18db35cacded44fb59c..096d5560bbb1e0251b544345a9ab9555342c79a7 100644
--- a/temper/temper-lib-zone/zone-names-es-03/zone-names-es-03.module.ts
+++ b/temper/temper-lib-zone/zone-names-es-03/zone-names-es-03.module.ts
@@ -9,7 +9,7 @@ export const zoneNamesEs03 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibZone v8.98 states.",
+      statement: "These rows are the rows upstream LibZone v8.98 states.",
     },
   ],
 } as const satisfies Module
