runsWarrants: false
diff --git a/pages/indexes/answering/index-answering.module.ts b/pages/indexes/answering/index-answering.module.ts
index d096a2972b42da1d62b6c4d2524ecdace780c63a..cff5efd0bd32bacc05984789ca1148fba18bd388 100644
--- a/pages/indexes/answering/index-answering.module.ts
+++ b/pages/indexes/answering/index-answering.module.ts
@@ -47,7 +47,8 @@ export const indexAnswering = {
     },
     {
       invariantKind: "departure",
-      statement: "A question here takes what the reader beneath takes besides what is bound here.",
+      statement:
+        "A question here takes the arguments the reader beneath takes besides the arguments bound here.",
     },
     {
       invariantKind: "departure",
@@ -55,7 +56,7 @@ export const indexAnswering = {
     },
     {
       invariantKind: "departure",
-      statement: "An answer here is what the reader beneath answers with that reading.",
+      statement: "An answer here is the answer the reader beneath gives with that reading.",
     },
     {
       invariantKind: "departure",
