runsWarrants: false
diff --git a/command-system/commands/properties/help-notes.text-property.ts b/command-system/commands/properties/help-notes.text-property.ts
index 1e6fd93ffc5112012413671ecc2acdd3163684b6..d112ade0bb321bdf7b867c4f7a33199fa1dbb2f2 100644
--- a/command-system/commands/properties/help-notes.text-property.ts
+++ b/command-system/commands/properties/help-notes.text-property.ts
@@ -17,7 +17,7 @@ export const helpNotes = {
     },
     {
       invariantKind: "departure",
-      statement: "The lines are shown under the list of what a command takes.",
+      statement: "The lines are shown under the list of a command's arguments.",
     },
     {
       invariantKind: "departure",
