runsWarrants: false
diff --git a/seat-system/recipient-resolving/recipient-resolver-running/recipient-resolver-running.module.ts b/seat-system/recipient-resolving/recipient-resolver-running/recipient-resolver-running.module.ts
index fbc88c668566827e705524854daea7b6d8f98288..0f69cf4fd8fd91884656c3854d808ced88d1f8a1 100644
--- a/seat-system/recipient-resolving/recipient-resolver-running/recipient-resolver-running.module.ts
+++ b/seat-system/recipient-resolving/recipient-resolver-running/recipient-resolver-running.module.ts
@@ -31,8 +31,7 @@ export const recipientResolverRunning = {
     },
     {
       invariantKind: "departure",
-      statement:
-        "What the recipient resolver is configured with is stated when that resolver starts.",
+      statement: "The recipient resolver's configuration is stated when that resolver starts.",
     },
     {
       invariantKind: "departure",
