runsWarrants: false
diff --git a/temper/temper-upstream-data/zone-upstream-port/zone-upstream-port.module.ts b/temper/temper-upstream-data/zone-upstream-port/zone-upstream-port.module.ts
index fe81f3dca73ecb227d9962cf8e25bf8c81d1a149..782e7ea6e7e031e9cf6cfc4cccc1c4ce26f4f8a7 100644
--- a/temper/temper-upstream-data/zone-upstream-port/zone-upstream-port.module.ts
+++ b/temper/temper-upstream-data/zone-upstream-port/zone-upstream-port.module.ts
@@ -14,7 +14,8 @@ export const zoneUpstreamPort = {
     },
     {
       invariantKind: "departure",
-      statement: "The machine is given the ESO stubs the upstream file calls before it is run.",
+      statement:
+        "The machine is given the ESO stubs the upstream file calls before that file is run.",
     },
     {
       invariantKind: "departure",
