runsWarrants: false
diff --git a/temper/temper-lib-map-data/map-data-zones-02/map-data-zones-02.module.ts b/temper/temper-lib-map-data/map-data-zones-02/map-data-zones-02.module.ts
index daa41bc9d3825fec3c51cfa6a59fbe603dd27944..1864ef774eac471b92ae81cf2cdae7550f7f21f8 100644
--- a/temper/temper-lib-map-data/map-data-zones-02/map-data-zones-02.module.ts
+++ b/temper/temper-lib-map-data/map-data-zones-02/map-data-zones-02.module.ts
@@ -9,7 +9,7 @@ export const mapDataZones02 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibMapData v1.21 states.",
+      statement: "These rows are the rows upstream LibMapData v1.21 states.",
     },
   ],
 } as const satisfies Module
