runsWarrants: false
diff --git a/temper/temper-lib-character-knowledge/knowledge-base-data-recipes/knowledge-base-data-recipes.module.ts b/temper/temper-lib-character-knowledge/knowledge-base-data-recipes/knowledge-base-data-recipes.module.ts
index 73ae263729e9d72b3427e49dd1e2e6e7d90eb96f..4e238c654f999783508bb381a8919f704cee4fb2 100644
--- a/temper/temper-lib-character-knowledge/knowledge-base-data-recipes/knowledge-base-data-recipes.module.ts
+++ b/temper/temper-lib-character-knowledge/knowledge-base-data-recipes/knowledge-base-data-recipes.module.ts
@@ -9,7 +9,7 @@ export const knowledgeBaseDataRecipes = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These ids are what upstream BaseData for API 101050 states.",
+      statement: "These ids are the ids upstream BaseData for API 101050 states.",
     },
   ],
 } as const satisfies Module
