runsWarrants: false
diff --git a/temper/temper-character/players/properties/platform.select-property.ts b/temper/temper-character/players/properties/platform.select-property.ts
index 4beea7e3665f318e9966bad893b8180010c424e2..ec41cc67a8aca4714eade9764daaa6bbe8704a4a 100644
--- a/temper/temper-character/players/properties/platform.select-property.ts
+++ b/temper/temper-character/players/properties/platform.select-property.ts
@@ -10,7 +10,7 @@ export const platform = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "The machines are named as The Elder Scrolls Online names them.",
+      statement: "The machines are named as The Elder Scrolls Online names those machines.",
     },
   ],
 } as const satisfies SelectProperty
