runsWarrants: false
diff --git a/temper/temper-items-rules-core/inventory-rule-matcher-types/inventory-rule-matcher-types.module.ts b/temper/temper-items-rules-core/inventory-rule-matcher-types/inventory-rule-matcher-types.module.ts
index ae4fd9242bbccc48baa25d36f4297f678de7f789..0c19faa98f6f4a0ac1dd8df576f4a107abec35bf 100644
--- a/temper/temper-items-rules-core/inventory-rule-matcher-types/inventory-rule-matcher-types.module.ts
+++ b/temper/temper-items-rules-core/inventory-rule-matcher-types/inventory-rule-matcher-types.module.ts
@@ -9,7 +9,7 @@ export const inventoryRuleMatcherTypes = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "An affected item carries where the item lies as well as what the item is.",
+      statement: "An affected item carries where the item lies as well as the item's own data.",
     },
     {
       invariantKind: "departure",
