runsWarrants: false
diff --git a/temper/temper-build-deploy-checks/ts-node-shapes/ts-node-shapes.module.ts b/temper/temper-build-deploy-checks/ts-node-shapes/ts-node-shapes.module.ts
index 557920b0889b1ac3ec04b4cfc1a1b43a7b25fbfd..3597557e921c0f9e8088b0790708a34788f1d513 100644
--- a/temper/temper-build-deploy-checks/ts-node-shapes/ts-node-shapes.module.ts
+++ b/temper/temper-build-deploy-checks/ts-node-shapes/ts-node-shapes.module.ts
@@ -9,7 +9,7 @@ export const tsNodeShapes = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "A node's name is read off the node or off what the node is bound to.",
+      statement: "A node's name is read off the node or off the declaration the node is bound to.",
     },
     {
       invariantKind: "constraint",
