runsWarrants: false
diff --git a/temper/temper-watcher/watcher-version/watcher-version.module.ts b/temper/temper-watcher/watcher-version/watcher-version.module.ts
index e0f6a9d8fdd65d0ade60565b274a025150ec80df..fcf649b2331931ee1d97f62444217f6248d96211 100644
--- a/temper/temper-watcher/watcher-version/watcher-version.module.ts
+++ b/temper/temper-watcher/watcher-version/watcher-version.module.ts
@@ -26,7 +26,7 @@ export const watcherVersion = {
     },
     {
       invariantKind: "constraint",
-      statement: "Moving this file does not change what the bundler puts in.",
+      statement: "Moving this file does not change the commit the bundler puts in.",
     },
     {
       invariantKind: "absence",
