runsWarrants: false
diff --git a/named-for/name-rule/name-rule.module.ts b/named-for/name-rule/name-rule.module.ts
index 60bf17ac9beb8406421a7d496a46ebbf154915d9..9c50611eda70f790f664fa2191f260f489115bf0 100644
--- a/named-for/name-rule/name-rule.module.ts
+++ b/named-for/name-rule/name-rule.module.ts
@@ -22,7 +22,7 @@ export const nameRule = {
     {
       invariantKind: "departure",
       statement:
-        "A rule whose holes are all filled but which still fills nothing answers with all of them.",
+        "A rule whose holes are all filled but which still fills nothing answers with all of those holes.",
     },
   ],
 } as const satisfies Module
