runsWarrants: false
diff --git a/person-system/people/email/email-messages/email-message.page-type.ts b/person-system/people/email/email-messages/email-message.page-type.ts
index 6cb6db680a3dc1afbfce2f2d9f25fac33182a8fd..885ad521048871c33ea4e3c7e123b8342255afbc 100644
--- a/person-system/people/email/email-messages/email-message.page-type.ts
+++ b/person-system/people/email/email-messages/email-message.page-type.ts
@@ -13,7 +13,7 @@ export const emailMessage = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A message is what an email rule is matched against.",
+      statement: "An email rule is matched against a message.",
     },
     {
       invariantKind: "departure",
