runsWarrants: false
diff --git a/temper/temper-watcher/watcher-log-line/watcher-log-line.module.ts b/temper/temper-watcher/watcher-log-line/watcher-log-line.module.ts
index fa83a923a302c6e7437a51bd82e6b88298b9b2f4..7eb374079d285183fabe0bb8394e331acbac56e7 100644
--- a/temper/temper-watcher/watcher-log-line/watcher-log-line.module.ts
+++ b/temper/temper-watcher/watcher-log-line/watcher-log-line.module.ts
@@ -14,7 +14,7 @@ export const watcherLogLine = {
     },
     {
       invariantKind: "departure",
-      statement: "A reader says which of the two it is reading.",
+      statement: "A reader says which of the two that reader is reading.",
     },
     {
       invariantKind: "departure",
