runsWarrants: false
diff --git a/git/oid.domain.ts b/git/oid.domain.ts
index 7e82132e6bc08e1569c8e5821d6405a5b72618e2..dcee5c6e19e5f8f00164e42d728b281a5cff0987 100644
--- a/git/oid.domain.ts
+++ b/git/oid.domain.ts
@@ -8,7 +8,7 @@ export const oid = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "Every name for a git object id is `oid`, or a scoped form of it.",
+      statement: "Every name for a git object id is `oid`, or a scoped form of `oid`.",
     },
   ],
 } as const satisfies Domain
