runsWarrants: false
diff --git a/domains/parent/domain-parent.domain.ts b/domains/parent/domain-parent.domain.ts
index db4d4496f910c4fec787681c5e2929c4044ef9b3..2814f6f8bb69356f4b49bdf5c773ef93fe32e1ef 100644
--- a/domains/parent/domain-parent.domain.ts
+++ b/domains/parent/domain-parent.domain.ts
@@ -13,7 +13,7 @@ export const domainParent = {
     },
     {
       invariantKind: "departure",
-      statement: "A layer a domain is built on is not a parent of it.",
+      statement: "A layer a domain is built on is not a parent of that domain.",
     },
     {
       invariantKind: "departure",
