runsWarrants: false
diff --git a/utils-run/running/running.module.ts b/utils-run/running/running.module.ts
index e594f95faa53bfb2742321e156c99beb807d4cd8..3687b918b8eb621d9dc189c724ab22679622c915 100644
--- a/utils-run/running/running.module.ts
+++ b/utils-run/running/running.module.ts
@@ -31,7 +31,7 @@ export const running = {
     },
     {
       invariantKind: "departure",
-      statement: "A process inherits the environment of the one that started it.",
+      statement: "A process inherits the environment of the one that started that process.",
     },
     {
       invariantKind: "departure",
