runsWarrants: false
diff --git a/temper/temper-watcher/watcher-export-tasks/watcher-export-tasks.module.ts b/temper/temper-watcher/watcher-export-tasks/watcher-export-tasks.module.ts
index a2d9af4292a5c400a849f0fffd4b9d11e5c57fbe..5aaa3ddf6dd25eee3e48d8457cff0bd057fdbbe7 100644
--- a/temper/temper-watcher/watcher-export-tasks/watcher-export-tasks.module.ts
+++ b/temper/temper-watcher/watcher-export-tasks/watcher-export-tasks.module.ts
@@ -60,7 +60,7 @@ export const watcherExportTasks = {
     },
     {
       invariantKind: "absence",
-      statement: "Nothing here decides what a character's priority is.",
+      statement: "Nothing here decides a character's priority.",
     },
     {
       invariantKind: "absence",
