runsWarrants: false
diff --git a/changes/workflow-templates/properties/declaration.file-property.ts b/changes/workflow-templates/properties/declaration.file-property.ts
index 011a7d696b54b0105cbba245d9df52a7f63a8dd8..8f063c6830c5e04ef36c5b47e8c9d0ba64d06dfd 100644
--- a/changes/workflow-templates/properties/declaration.file-property.ts
+++ b/changes/workflow-templates/properties/declaration.file-property.ts
@@ -15,7 +15,7 @@ export const declaration = {
     },
     {
       invariantKind: "departure",
-      statement: "A declaration states the steps, and the page states what those steps are for.",
+      statement: "A declaration states the steps, and the page states those steps' purpose.",
     },
     {
       invariantKind: "departure",
