runsWarrants: false
diff --git a/commands/move/listing/move-listing.module.ts b/commands/move/listing/move-listing.module.ts
index 5602530942a891c8393e44d4c6e091984dfc3898..d3234d2d3142beab4292491406a5658541a03466 100644
--- a/commands/move/listing/move-listing.module.ts
+++ b/commands/move/listing/move-listing.module.ts
@@ -39,7 +39,7 @@ export const moveListing = {
     },
     {
       invariantKind: "absence",
-      statement: "Nothing here knows what a part means.",
+      statement: "Nothing here knows the meaning of a part.",
     },
   ],
 } as const satisfies Module
