runsWarrants: false
diff --git a/alan/tracking/daily/eso-days/eso-day.page-type.ts b/alan/tracking/daily/eso-days/eso-day.page-type.ts
index 36c1f0934a1b1eeb6215eaaa4800c68440f19a43..29c1810ccea9317d70018a8eef92d162ba934fe9 100644
--- a/alan/tracking/daily/eso-days/eso-day.page-type.ts
+++ b/alan/tracking/daily/eso-days/eso-day.page-type.ts
@@ -50,7 +50,7 @@ export const esoDay = {
     },
     {
       invariantKind: "departure",
-      statement: "An ESO day is slugged `eso-day-` before the day it is of.",
+      statement: "An ESO day is slugged `eso-day-` before the day that ESO day is of.",
     },
     {
       invariantKind: "departure",
