runsWarrants: false
diff --git a/checks/code-checks/pages/no-refused-syntax/syntax-rules/no-void-return/no-void-return.syntax-rule.ts b/checks/code-checks/pages/no-refused-syntax/syntax-rules/no-void-return/no-void-return.syntax-rule.ts
index ccb2c4115a8ee0e5741a53135fc4fc879f639a7b..3ec04467cf7e5ce099f960cf2cfaadd4d8a76860 100644
--- a/checks/code-checks/pages/no-refused-syntax/syntax-rules/no-void-return/no-void-return.syntax-rule.ts
+++ b/checks/code-checks/pages/no-refused-syntax/syntax-rules/no-void-return/no-void-return.syntax-rule.ts
@@ -98,7 +98,7 @@ export const noVoidReturn = {
     },
     {
       invariantKind: "departure",
-      statement: "It is no type at all.",
+      statement: "`void` used as an operator on an expression is no type at all.",
     },
     {
       invariantKind: "gap",
