runsWarrants: false
diff --git a/command-system/commands/lint/lint.command.ts b/command-system/commands/lint/lint.command.ts
index 0d8219e58a5e2bba3a4e1a97ab0d2dec56154dbd..4c5a9349db3446a6f2cf33ffe58d2a8b1384d5c9 100644
--- a/command-system/commands/lint/lint.command.ts
+++ b/command-system/commands/lint/lint.command.ts
@@ -36,7 +36,7 @@ export const lint = {
     },
     {
       invariantKind: "absence",
-      statement: "What is checked is what the linter is configured by.",
+      statement: "The rules checked are the rules the linter is configured by.",
     },
   ],
 } as const satisfies Command
