runsWarrants: false
diff --git a/temper/temper-crafting-addon/temper-crafting-addon.eso-addon.ts b/temper/temper-crafting-addon/temper-crafting-addon.eso-addon.ts
index 9f3c57134045121369cfbbf6357b5ddc9ac5823a..309ce09aa8b7e494f88b95756bfb4db200a663aa 100644
--- a/temper/temper-crafting-addon/temper-crafting-addon.eso-addon.ts
+++ b/temper/temper-crafting-addon/temper-crafting-addon.eso-addon.ts
@@ -215,7 +215,7 @@ export const temperCraftingAddon = {
     },
     {
       invariantKind: "departure",
-      statement: "What a character knows is asked of LibCharacterKnowledge.",
+      statement: "A character's knowledge is asked of LibCharacterKnowledge.",
     },
     {
       invariantKind: "departure",
