runsWarrants: false
diff --git a/ki/ki-episodes/ki-episode.page-type.ts b/ki/ki-episodes/ki-episode.page-type.ts
index de875107216f428fac32f28c179dd50af05821d3..9710149df22d5d062882fae4a86ce5ae7b41b7b6 100644
--- a/ki/ki-episodes/ki-episode.page-type.ts
+++ b/ki/ki-episodes/ki-episode.page-type.ts
@@ -13,7 +13,7 @@ export const kiEpisode = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "An episode of Ki's names the season it is part of.",
+      statement: "An episode of Ki's names the season that episode is part of.",
     },
     {
       invariantKind: "departure",
