runsWarrants: false
diff --git a/temper/temper-items-rules-matcher/inventory-rule-matcher/inventory-rule-matcher.module.ts b/temper/temper-items-rules-matcher/inventory-rule-matcher/inventory-rule-matcher.module.ts
index 4aa2ff9b97f49c2d74f698d6859fa9830dd5554d..4b711cb6cc670aa48a973220f287e17fab46d15b 100644
--- a/temper/temper-items-rules-matcher/inventory-rule-matcher/inventory-rule-matcher.module.ts
+++ b/temper/temper-items-rules-matcher/inventory-rule-matcher/inventory-rule-matcher.module.ts
@@ -13,7 +13,7 @@ export const inventoryRuleMatcher = {
     },
     {
       invariantKind: "departure",
-      statement: "What one rule leaves over is offered to the rule below.",
+      statement: "The items one rule leaves over are offered to the rule below.",
     },
     {
       invariantKind: "departure",
