runsWarrants: false
diff --git a/temper/temper-addon-data/generate-addon-data/generate-addon-data.module.ts b/temper/temper-addon-data/generate-addon-data/generate-addon-data.module.ts
index c83875890920af1e0cb8fe76f70be97cc33babb5..153dd5c95d075346fe13a63e9a137d9a2f41cf93 100644
--- a/temper/temper-addon-data/generate-addon-data/generate-addon-data.module.ts
+++ b/temper/temper-addon-data/generate-addon-data/generate-addon-data.module.ts
@@ -10,7 +10,8 @@ export const generateAddonData = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "The output directories are made before any file is written into them.",
+      statement:
+        "The output directories are made before any file is written into those directories.",
     },
     {
       invariantKind: "departure",
