diff --git a/code-system/file-kind/file-kind.module.ts b/code-system/file-kind/file-kind.module.ts
index 620fb9e6c10e545ae64a5b12c25eef7ed30badf3..8c13aab646837f6e772a3acc7c5e9e08675cbea7 100644
--- a/code-system/file-kind/file-kind.module.ts
+++ b/code-system/file-kind/file-kind.module.ts
@@ -13,7 +13,7 @@ export const fileKind = {
     },
     {
       invariantKind: "departure",
-      statement: "A whole basename is read before any extension of it is.",
+      statement: "A whole basename is read before any extension of that basename is.",
     },
     {
       invariantKind: "departure",
@@ -49,7 +49,7 @@ export const fileKind = {
     },
     {
       invariantKind: "departure",
-      statement: "A systemd timer stands on the service it starts.",
+      statement: "A systemd timer stands on the service that timer starts.",
     },
     {
       invariantKind: "departure",
