runsWarrants: false
diff --git a/temper/temper-inventory-automation/automation-toggle-change/automation-toggle-change.module.ts b/temper/temper-inventory-automation/automation-toggle-change/automation-toggle-change.module.ts
index deea0a7a9905f8fdc73e80aaa81eebcadeb89f88..e3ec46a257d555bc159950c88bc038116bc6f369 100644
--- a/temper/temper-inventory-automation/automation-toggle-change/automation-toggle-change.module.ts
+++ b/temper/temper-inventory-automation/automation-toggle-change/automation-toggle-change.module.ts
@@ -42,7 +42,7 @@ export const automationToggleChange = {
     },
     {
       invariantKind: "departure",
-      statement: "A value of nothing takes the toggle away rather than switching it off.",
+      statement: "A value of nothing takes the toggle away rather than switching that toggle off.",
     },
     {
       invariantKind: "departure",
