runsWarrants: false
diff --git a/temper/temper-housing-addon/housing-publish/housing-publish.module.ts b/temper/temper-housing-addon/housing-publish/housing-publish.module.ts
index 23a3463379a5fef46b639dadcc7d8da90437875f..a8d01d05a24ecfef351daf550b8938cce71e7295 100644
--- a/temper/temper-housing-addon/housing-publish/housing-publish.module.ts
+++ b/temper/temper-housing-addon/housing-publish/housing-publish.module.ts
@@ -9,7 +9,7 @@ export const housingPublish = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "What other add-ons may read is published under one global name.",
+      statement: "The name and version other add-ons may read are published under one global name.",
     },
   ],
 } as const satisfies Module
