runsWarrants: false
diff --git a/command-system/fault-saying/fault-saying.module.ts b/command-system/fault-saying/fault-saying.module.ts
index 86c03a4af9cf2e957400de23cd3e34086d43531c..87f77ca1b0564b174dc598444fc3e10d9d4b2615 100644
--- a/command-system/fault-saying/fault-saying.module.ts
+++ b/command-system/fault-saying/fault-saying.module.ts
@@ -10,7 +10,7 @@ export const faultSaying = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "What is thrown is read for a message.",
+      statement: "A thrown thing is read for a message.",
     },
     {
       invariantKind: "departure",
