runsWarrants: false
diff --git a/check/akasha-check.domain.ts b/check/akasha-check.domain.ts
index fd07533a92d6ca6d8a781b49c57b5781ce864d70..08aec29089c3ef1a30502c4c88c76ecb53f868c4 100644
--- a/check/akasha-check.domain.ts
+++ b/check/akasha-check.domain.ts
@@ -9,7 +9,7 @@ export const akashaCheck = {
     {
       invariantKind: "stopgap",
       statement:
-        "No check exists only to test what the compiler tests apart from the one that runs it.",
+        "No check exists only to test what the compiler tests apart from the one that runs that compiler.",
     },
     {
       invariantKind: "gap",
