runsWarrants: false
diff --git a/temper/temper-lib-zone/zone-names-fr-00/zone-names-fr-00.module.ts b/temper/temper-lib-zone/zone-names-fr-00/zone-names-fr-00.module.ts
index 1c483c0e86d5aefd2e9adab42fd92e915eda847f..0936ae7971450ddce25a17b9e8b9ed650b792e70 100644
--- a/temper/temper-lib-zone/zone-names-fr-00/zone-names-fr-00.module.ts
+++ b/temper/temper-lib-zone/zone-names-fr-00/zone-names-fr-00.module.ts
@@ -9,7 +9,7 @@ export const zoneNamesFr00 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibZone v8.98 states.",
+      statement: "These rows are the rows upstream LibZone v8.98 states.",
     },
   ],
 } as const satisfies Module
