runsWarrants: false
diff --git a/service-system/cluster-services/properties/image.text-property.ts b/service-system/cluster-services/properties/image.text-property.ts
index 94d6b1e0d3f7f5dfdb3fc3c4e77b8164e54ff43b..465c17a5ded342d8c57f1945b90e6396b4230f96 100644
--- a/service-system/cluster-services/properties/image.text-property.ts
+++ b/service-system/cluster-services/properties/image.text-property.ts
@@ -18,7 +18,7 @@ export const image = {
     {
       invariantKind: "departure",
       statement:
-        "A workload running more than one container states the image of the one it is for.",
+        "A workload running more than one container states the image of the one that workload is for.",
     },
   ],
 } as const satisfies TextProperty
