runsWarrants: false
diff --git a/reminder-system/reminder-system.workspace-package.ts b/reminder-system/reminder-system.workspace-package.ts
index 9201f67afe06dc0e22c2e39fe84d56c7e872af5a..8e39a8c18458d7cad6a360a6e92d4e56a28c9a99 100644
--- a/reminder-system/reminder-system.workspace-package.ts
+++ b/reminder-system/reminder-system.workspace-package.ts
@@ -10,11 +10,11 @@ export const reminderSystem = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A reminder is sent by a clock rather than by whoever wrote the reminder.",
+      statement: "A reminder is sent by a clock rather than by the person who wrote the reminder.",
     },
     {
       invariantKind: "departure",
-      statement: "What a reminder is sent as is a page in akasha.",
+      statement: "A reminder is sent as a page in akasha.",
     },
     {
       invariantKind: "departure",
