runsWarrants: false
diff --git a/temper/temper-player-completion/completion-generic-checker-progress/completion-generic-checker-progress.module.ts b/temper/temper-player-completion/completion-generic-checker-progress/completion-generic-checker-progress.module.ts
index 359208c9efdebf6e1e6c4bdc6c9a65274eb8eb3a..635c9bd56ad0e50d1fd5cfd58c5430dd1ef4f108 100644
--- a/temper/temper-player-completion/completion-generic-checker-progress/completion-generic-checker-progress.module.ts
+++ b/temper/temper-player-completion/completion-generic-checker-progress/completion-generic-checker-progress.module.ts
@@ -14,7 +14,7 @@ export const completionGenericCheckerProgress = {
     },
     {
       invariantKind: "departure",
-      statement: "A leaf stating its own numbers is taken over counting it as one of one.",
+      statement: "A leaf stating its own numbers is taken over counting that leaf as one of one.",
     },
     {
       invariantKind: "departure",
