runsWarrants: false
diff --git a/pages/named-file-properties/named-file-property.page-type.ts b/pages/named-file-properties/named-file-property.page-type.ts
index c81a8a3f8feaf40686981b5a9819614638f43913..f15491763a329b1c549e349335409696f496ecec 100644
--- a/pages/named-file-properties/named-file-property.page-type.ts
+++ b/pages/named-file-properties/named-file-property.page-type.ts
@@ -26,7 +26,7 @@ export const namedFileProperty = {
     },
     {
       invariantKind: "departure",
-      statement: "A page claims a named file by stating the property holding it.",
+      statement: "A page claims a named file by stating the property holding that file.",
     },
     {
       invariantKind: "departure",
