runsWarrants: false
diff --git a/pages/queries/properties/asks-of-slug.text-property.ts b/pages/queries/properties/asks-of-slug.text-property.ts
index 7c2ce7b358a180b2b66e1ae9894f5e808b86607b..2bcad4d489db226fa8a5df9620f56175a90f1976 100644
--- a/pages/queries/properties/asks-of-slug.text-property.ts
+++ b/pages/queries/properties/asks-of-slug.text-property.ts
@@ -13,7 +13,8 @@ export const asksOfSlug = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A query names one page type and asks of every page type extending it.",
+      statement:
+        "A query names one page type and asks of every page type extending that page type.",
     },
     {
       invariantKind: "departure",
