runsWarrants: false
diff --git a/alan/tracking/to-dos/to-do.page-type.ts b/alan/tracking/to-dos/to-do.page-type.ts
index 453ead0cbbe2f6949d27f983e5a9121d2d0c1098..6e4ed0dfd733ee877a89506821fc11ba81118140 100644
--- a/alan/tracking/to-dos/to-do.page-type.ts
+++ b/alan/tracking/to-dos/to-do.page-type.ts
@@ -90,7 +90,7 @@ export const toDo = {
     },
     {
       invariantKind: "departure",
-      statement: "A to-do's body is what doing that to-do takes.",
+      statement: "A to-do's body is the work doing that to-do takes.",
     },
     {
       invariantKind: "departure",
