runsWarrants: false
diff --git a/domains/properties/definition.text-property.ts b/domains/properties/definition.text-property.ts
index 3dbb01e8f77a6d246de0b69928295b61f0cefe83..18d8d5a6365d273ff112b8aea8f8938119c50ea2 100644
--- a/domains/properties/definition.text-property.ts
+++ b/domains/properties/definition.text-property.ts
@@ -29,7 +29,8 @@ export const definition = {
     },
     {
       invariantKind: "departure",
-      statement: "Where a second concern is needed to cover the area it is more than one domain.",
+      statement:
+        "Where a second concern is needed to cover the area, that area is more than one domain.",
     },
     {
       invariantKind: "departure",
