runsWarrants: false
diff --git a/instrument/instrument.domain.ts b/instrument/instrument.domain.ts
index 26443807831ccdc90e06b4c4d770f7c1266ce78b..670f8c66061b96058da45c0107154d52dc60bdb4 100644
--- a/instrument/instrument.domain.ts
+++ b/instrument/instrument.domain.ts
@@ -14,7 +14,7 @@ export const instrument = {
     },
     {
       invariantKind: "departure",
-      statement: "An instrument never changes what the instrument was run on.",
+      statement: "An instrument never changes the subject the instrument was run on.",
     },
     {
       invariantKind: "departure",
