runsWarrants: false
diff --git a/infrastructure/inference-commands/inference-active/inference-active.command.ts b/infrastructure/inference-commands/inference-active/inference-active.command.ts
index 256806f8074bca3aa8db8da558f28a3e50e3034b..df2711d7e5a11bd1a547f23ddfa8e83d1da01a35 100644
--- a/infrastructure/inference-commands/inference-active/inference-active.command.ts
+++ b/infrastructure/inference-commands/inference-active/inference-active.command.ts
@@ -12,7 +12,7 @@ export const inferenceActive = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "What is resident is read off the cop.",
+      statement: "The resident services are read off the cop.",
     },
     {
       invariantKind: "departure",
