runsWarrants: false
diff --git a/temper/temper-lib-sets/lib-sets-set-text-set-text-fields/lib-sets-set-text-set-text-fields.module.ts b/temper/temper-lib-sets/lib-sets-set-text-set-text-fields/lib-sets-set-text-set-text-fields.module.ts
index 441f73db9693939171a662ea83c13f92340515f1..3671846900bdffea4462ece93539b4270fc52527 100644
--- a/temper/temper-lib-sets/lib-sets-set-text-set-text-fields/lib-sets-set-text-set-text-fields.module.ts
+++ b/temper/temper-lib-sets/lib-sets-set-text-set-text-fields/lib-sets-set-text-set-text-fields.module.ts
@@ -9,7 +9,8 @@ export const libSetsSetTextSetTextFields = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "A piece is skipped when neither the settings nor the pattern asks for it.",
+      statement:
+        "A piece is skipped when neither the settings nor the pattern asks for that piece.",
     },
     {
       invariantKind: "constraint",
