diff --git a/temper/temper-characters-equipment/weapon-enchants/weapon-enchants.module.ts b/temper/temper-characters-equipment/weapon-enchants/weapon-enchants.module.ts
index db5beef29957c2beef5b43788aa9a77639ec0d98..ac04e4f9f15dd4a44c8411c8c68ec0e5ebd715cd 100644
--- a/temper/temper-characters-equipment/weapon-enchants/weapon-enchants.module.ts
+++ b/temper/temper-characters-equipment/weapon-enchants/weapon-enchants.module.ts
@@ -23,9 +23,5 @@ export const weaponEnchants = {
       invariantKind: "upkeep",
       statement: "The generator writes this table outside akasha.",
     },
-    {
-      invariantKind: "upkeep",
-      statement: "Both copies of this table move together.",
-    },
   ],
 } as const satisfies Module
