runsWarrants: false
diff --git a/pages/file-name/page-file-name.module.ts b/pages/file-name/page-file-name.module.ts
index 76ce2d57d49dcdd73c49ceae4411ad21620374d1..90de41fae7a17e5250aceab0ee46556292cec4ba 100644
--- a/pages/file-name/page-file-name.module.ts
+++ b/pages/file-name/page-file-name.module.ts
@@ -23,11 +23,11 @@ export const pageFileName = {
     },
     {
       invariantKind: "departure",
-      statement: "What follows the page type is a list of sections.",
+      statement: "A list of sections follows the page type.",
     },
     {
       invariantKind: "departure",
-      statement: "The last part is what the file holds.",
+      statement: "The last part is the form the file holds.",
     },
     {
       invariantKind: "departure",
