runsWarrants: false
diff --git a/command-system/commands/move/repointing/move-repointing.module.ts b/command-system/commands/move/repointing/move-repointing.module.ts
index b9232a5dbb97f080a0861c498124039a8daa1d01..8859e2d9775795b60ba6e4602b9360b935267ba1 100644
--- a/command-system/commands/move/repointing/move-repointing.module.ts
+++ b/command-system/commands/move/repointing/move-repointing.module.ts
@@ -23,7 +23,7 @@ export const moveRepointing = {
     },
     {
       invariantKind: "departure",
-      statement: "What a package's name reaches moves without that name changing.",
+      statement: "The path a package's name reaches moves without that name changing.",
     },
     {
       invariantKind: "departure",
