runsWarrants: false
diff --git a/temper/temper-lib-zone/zone-names-ru-00/zone-names-ru-00.module.ts b/temper/temper-lib-zone/zone-names-ru-00/zone-names-ru-00.module.ts
index 5c73a9e93e849086bbf12610919a202648677313..7c9fcdbf37974305a8ac401ba12c5409c0551724 100644
--- a/temper/temper-lib-zone/zone-names-ru-00/zone-names-ru-00.module.ts
+++ b/temper/temper-lib-zone/zone-names-ru-00/zone-names-ru-00.module.ts
@@ -9,7 +9,7 @@ export const zoneNamesRu00 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibZone v8.98 states.",
+      statement: "These rows are the rows upstream LibZone v8.98 states.",
     },
   ],
 } as const satisfies Module
