runsWarrants: false
diff --git a/temper/temper-lib-zone/zone-names-de-02/zone-names-de-02.module.ts b/temper/temper-lib-zone/zone-names-de-02/zone-names-de-02.module.ts
index 2553cec650b2827661f9ea6461bfb6c2ee4db3bd..2a4231676816551377366f5d18e2d5aad507c952 100644
--- a/temper/temper-lib-zone/zone-names-de-02/zone-names-de-02.module.ts
+++ b/temper/temper-lib-zone/zone-names-de-02/zone-names-de-02.module.ts
@@ -9,7 +9,7 @@ export const zoneNamesDe02 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibZone v8.98 states.",
+      statement: "These rows are the rows upstream LibZone v8.98 states.",
     },
   ],
 } as const satisfies Module
