runsWarrants: false
diff --git a/editor-extension/tree-filter/tree-filter.module.ts b/editor-extension/tree-filter/tree-filter.module.ts
index 100a7b53dc6c5b07e94835d779ab940a9a76186c..30a85c29040a157ac5e0a1d897aeaf9b619db5e6 100644
--- a/editor-extension/tree-filter/tree-filter.module.ts
+++ b/editor-extension/tree-filter/tree-filter.module.ts
@@ -50,7 +50,7 @@ export const treeFilter = {
     },
     {
       invariantKind: "absence",
-      statement: "Nothing here knows what makes a node match.",
+      statement: "Nothing here knows why a node matches.",
     },
     {
       invariantKind: "absence",
