runsWarrants: false
diff --git a/pages/body/page-body.module.ts b/pages/body/page-body.module.ts
index 46486e588141737f042807d6c5c392de0a80dcb3..a5ee58df6174dd9a3dda7d0be7afb1a3f3099097 100644
--- a/pages/body/page-body.module.ts
+++ b/pages/body/page-body.module.ts
@@ -35,7 +35,7 @@ export const pageBody = {
     },
     {
       invariantKind: "departure",
-      statement: "Only what JSON holds may be a value.",
+      statement: "Only data JSON holds may be a value.",
     },
     {
       invariantKind: "departure",
