diff --git a/pages/types/properties/properties.record-property.ts b/pages/types/properties/properties.record-property.ts
index f181fe6035667063072f7e4d91115c3f2f1a6235..08ee347436a72cc45b9b8bb5b5b0ce47f118b4ae 100644
--- a/pages/types/properties/properties.record-property.ts
+++ b/pages/types/properties/properties.record-property.ts
@@ -101,7 +101,7 @@ export const properties = {
     {
       invariantKind: "departure",
       statement:
-        "A value stands in the commit unless the declaration carrying it says it does not.",
+        "A value stands in the commit unless the declaration carrying that value says that value does not.",
     },
     {
       invariantKind: "departure",
