runsWarrants: false
diff --git a/code-system/ios-programs/properties/target-name.text-property.ts b/code-system/ios-programs/properties/target-name.text-property.ts
index 338c0151dba749e61b1588e3f73cda0b02c629d4..f1763ed5264894702e457bf37adbb39723283900 100644
--- a/code-system/ios-programs/properties/target-name.text-property.ts
+++ b/code-system/ios-programs/properties/target-name.text-property.ts
@@ -14,7 +14,7 @@ export const targetName = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "What a program is built under is what the built bundle is named for.",
+      statement: "The name a program is built under is the name the built bundle is named for.",
     },
   ],
 } as const satisfies TextProperty
