runsWarrants: false
diff --git a/pages/file-name/page-file-name.module.ts b/pages/file-name/page-file-name.module.ts
index 76ce2d57d49dcdd73c49ceae4411ad21620374d1..f21260c500994d4c41617a105fb0cce158268ce6 100644
--- a/pages/file-name/page-file-name.module.ts
+++ b/pages/file-name/page-file-name.module.ts
@@ -23,7 +23,7 @@ export const pageFileName = {
     },
     {
       invariantKind: "departure",
-      statement: "What follows the page type is a list of sections.",
+      statement: "A list of sections follows the page type.",
     },
     {
       invariantKind: "departure",
