runsWarrants: false
diff --git a/domains/purpose/domain-purpose.domain.ts b/domains/purpose/domain-purpose.domain.ts
index e4e805f8a1cc4dd61c5152daa738fe29067c9783..8bd74050d3b1aa7c01911dd5e49bc73d5761b243 100644
--- a/domains/purpose/domain-purpose.domain.ts
+++ b/domains/purpose/domain-purpose.domain.ts
@@ -8,7 +8,7 @@ export const domainPurpose = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "What only helps another purpose is not a purpose.",
+      statement: "A thing that only helps another purpose is not a purpose.",
     },
   ],
 } as const satisfies Domain
