runsWarrants: false
diff --git a/temper/temper-lib-zone/zone-names-pl-00/zone-names-pl-00.module.ts b/temper/temper-lib-zone/zone-names-pl-00/zone-names-pl-00.module.ts
index 3498888eb1795e913243d1d431d9a9cf3e15458f..10303fb3c58d2e1be19bf814ddaa09e2c7c23a24 100644
--- a/temper/temper-lib-zone/zone-names-pl-00/zone-names-pl-00.module.ts
+++ b/temper/temper-lib-zone/zone-names-pl-00/zone-names-pl-00.module.ts
@@ -9,7 +9,7 @@ export const zoneNamesPl00 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibZone v8.98 states.",
+      statement: "These rows are the rows upstream LibZone v8.98 states.",
     },
   ],
 } as const satisfies Module
