runsWarrants: false
diff --git a/collections/units/unit.page-type.ts b/collections/units/unit.page-type.ts
index 6bf6ef6653b5e3a70363ac90aacbbd23ab801992..864cf907c6bca406d29f6e8e575a2eb1b21815d0 100644
--- a/collections/units/unit.page-type.ts
+++ b/collections/units/unit.page-type.ts
@@ -25,7 +25,7 @@ export const unit = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A unit says what a unit is worth in words.",
+      statement: "A unit says its worth in words.",
     },
     {
       invariantKind: "departure",
