runsWarrants: false
diff --git a/alan/music/spotify/client/spotify-client.module.ts b/alan/music/spotify/client/spotify-client.module.ts
index 3db5cd54955febcc1294b3c631650308880124e6..d66da0ca35d06f036291010ed537114f75b9ea2f 100644
--- a/alan/music/spotify/client/spotify-client.module.ts
+++ b/alan/music/spotify/client/spotify-client.module.ts
@@ -62,7 +62,8 @@ export const spotifyClient = {
     },
     {
       invariantKind: "departure",
-      statement: "A path opening with a scheme is called as it is rather than under the base URL.",
+      statement:
+        "A path opening with a scheme is called as the path is rather than under the base URL.",
     },
     {
       invariantKind: "departure",
