runsWarrants: false
diff --git a/pages/beside/page-beside.module.ts b/pages/beside/page-beside.module.ts
index 49597d75338c0ee077922cdc258dd5d4abd1ecfb..889b21d678bb61c4a625a93815274f3b8b473a14 100644
--- a/pages/beside/page-beside.module.ts
+++ b/pages/beside/page-beside.module.ts
@@ -28,7 +28,7 @@ export const pageBeside = {
     },
     {
       invariantKind: "departure",
-      statement: "What is answered is sorted.",
+      statement: "The files answered are sorted.",
     },
     {
       invariantKind: "departure",
