diff --git a/ki/ki-franchises/ki-franchise.page-type.ts b/ki/ki-franchises/ki-franchise.page-type.ts
index 7bbe1e04f77d3c8ffb72f42708cc2492c0f4572c..e7d416907cdfa205575145c5f839156237a52aaa 100644
--- a/ki/ki-franchises/ki-franchise.page-type.ts
+++ b/ki/ki-franchises/ki-franchise.page-type.ts
@@ -13,11 +13,11 @@ export const kiFranchise = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A franchise of Ki's names the shows it takes in.",
+      statement: "A franchise of Ki's names the shows that franchise takes in.",
     },
     {
       invariantKind: "departure",
-      statement: "A franchise of Ki's names the movies it takes in.",
+      statement: "A franchise of Ki's names the movies that franchise takes in.",
     },
     {
       invariantKind: "departure",
