runsWarrants: false
diff --git a/temper/temper-lib-zone/zone-geo-data-00/zone-geo-data-00.module.ts b/temper/temper-lib-zone/zone-geo-data-00/zone-geo-data-00.module.ts
index 8c4b723acb8c591bcf73134e3459d8013340df9b..a295980457b68d2423d8bc3810bbd0b2d35751d5 100644
--- a/temper/temper-lib-zone/zone-geo-data-00/zone-geo-data-00.module.ts
+++ b/temper/temper-lib-zone/zone-geo-data-00/zone-geo-data-00.module.ts
@@ -9,7 +9,7 @@ export const zoneGeoData00 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibZone v8.98 states.",
+      statement: "These rows are the rows upstream LibZone v8.98 states.",
     },
   ],
 } as const satisfies Module
