runsWarrants: false
diff --git a/temper/temper-lib-zone/zone-names-pl-02/zone-names-pl-02.module.ts b/temper/temper-lib-zone/zone-names-pl-02/zone-names-pl-02.module.ts
index 57987635072079af51a8e79c1d50c177d2106b0d..b10802ff47763b8d5ba03c6db60f4b05ca77a394 100644
--- a/temper/temper-lib-zone/zone-names-pl-02/zone-names-pl-02.module.ts
+++ b/temper/temper-lib-zone/zone-names-pl-02/zone-names-pl-02.module.ts
@@ -9,7 +9,7 @@ export const zoneNamesPl02 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibZone v8.98 states.",
+      statement: "These rows are the rows upstream LibZone v8.98 states.",
     },
   ],
 } as const satisfies Module
