runsWarrants: false
diff --git a/temper/temper-upstream-data/map-data-upstream-port/map-data-upstream-port.module.ts b/temper/temper-upstream-data/map-data-upstream-port/map-data-upstream-port.module.ts
index f2205a34af54f8cc57011eab5d24f7d1f5b7cb9a..636fbc86e4fad4a4e030f98a5870f55c43d46220 100644
--- a/temper/temper-upstream-data/map-data-upstream-port/map-data-upstream-port.module.ts
+++ b/temper/temper-upstream-data/map-data-upstream-port/map-data-upstream-port.module.ts
@@ -22,7 +22,7 @@ export const mapDataUpstreamPort = {
     {
       invariantKind: "departure",
       statement:
-        "The pseudo map indices are gathered from the globals the upstream file names them on.",
+        "The pseudo map indices are gathered from the globals the upstream file names those indices on.",
     },
     {
       invariantKind: "departure",
