runsWarrants: false
diff --git a/imessage/commands/imessage-contacts/imessage-contacts.command.ts b/imessage/commands/imessage-contacts/imessage-contacts.command.ts
index d251a5833cdae31185dbf57f34e01452b03bbf43..6b3ad2488dac8bcff75865721aee0829f5b275c0 100644
--- a/imessage/commands/imessage-contacts/imessage-contacts.command.ts
+++ b/imessage/commands/imessage-contacts/imessage-contacts.command.ts
@@ -21,7 +21,7 @@ export const imessageContacts = {
   invariants: [
     {
       invariantKind: "departure",
-      statement: "A name is matched whatever the case it was said in.",
+      statement: "A name is matched whatever the case that name was said in.",
     },
     {
       invariantKind: "departure",
