runsWarrants: false
diff --git a/temper/temper-lib-map-data/map-data-index-02/map-data-index-02.module.ts b/temper/temper-lib-map-data/map-data-index-02/map-data-index-02.module.ts
index c2dee29d5e4078b133e28200f0ff03e2a302d19b..d3ecf88076e6d56f9c639a3df276c406a74afd27 100644
--- a/temper/temper-lib-map-data/map-data-index-02/map-data-index-02.module.ts
+++ b/temper/temper-lib-map-data/map-data-index-02/map-data-index-02.module.ts
@@ -9,7 +9,7 @@ export const mapDataIndex02 = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These rows are what upstream LibMapData v1.21 states.",
+      statement: "These rows are the rows upstream LibMapData v1.21 states.",
     },
   ],
 } as const satisfies Module
