runsWarrants: false
diff --git a/pages/unique-kinds/pages/part-of.unique-kind.ts b/pages/unique-kinds/pages/part-of.unique-kind.ts
index 7900cacc3fcae2aecb087abe269905ad1347a4c1..eef6de8eb8cf36c01b35a75367635a2b36e91275 100644
--- a/pages/unique-kinds/pages/part-of.unique-kind.ts
+++ b/pages/unique-kinds/pages/part-of.unique-kind.ts
@@ -12,7 +12,7 @@ export const partOf = {
     },
     {
       invariantKind: "departure",
-      statement: "A collection is part of what that collection names under `part-of-slugs`.",
+      statement: "A collection is part of the pages that collection names under `part-of-slugs`.",
     },
     {
       invariantKind: "departure",
