runsWarrants: false
diff --git a/temper/temper-lib-character-knowledge/knowledge-base-data-motifs/knowledge-base-data-motifs.module.ts b/temper/temper-lib-character-knowledge/knowledge-base-data-motifs/knowledge-base-data-motifs.module.ts
index 346d1e30a1ecfcef8f348b9b5daaab096479724b..e5476e17fcd9ed6066fe5e6ea3e085830671cdf0 100644
--- a/temper/temper-lib-character-knowledge/knowledge-base-data-motifs/knowledge-base-data-motifs.module.ts
+++ b/temper/temper-lib-character-knowledge/knowledge-base-data-motifs/knowledge-base-data-motifs.module.ts
@@ -9,7 +9,7 @@ export const knowledgeBaseDataMotifs = {
   invariants: [
     {
       invariantKind: "constraint",
-      statement: "These ids are what upstream BaseData for API 101050 states.",
+      statement: "These ids are the ids upstream BaseData for API 101050 states.",
     },
   ],
 } as const satisfies Module
