runsWarrants: false
diff --git a/temper/temper-watcher/watcher-updating/watcher-updating.module.ts b/temper/temper-watcher/watcher-updating/watcher-updating.module.ts
index 2b97d66f2c88333d91c9d4eda82e53e26005540a..e620dba8daca4a2f261df3e0f7a3a5f00e057bb6 100644
--- a/temper/temper-watcher/watcher-updating/watcher-updating.module.ts
+++ b/temper/temper-watcher/watcher-updating/watcher-updating.module.ts
@@ -90,11 +90,11 @@ export const watcherUpdating = {
     },
     {
       invariantKind: "departure",
-      statement: "A caller may hand in what fetches the deployed version.",
+      statement: "A caller may hand in the fetch answering the deployed version.",
     },
     {
       invariantKind: "departure",
-      statement: "A caller may hand in what fetches the executable.",
+      statement: "A caller may hand in the fetch answering the executable.",
     },
     {
       invariantKind: "departure",
