runsWarrants: false
diff --git a/graph/edges/graph-edge.page-type.ts b/graph/edges/graph-edge.page-type.ts
index 2f2c89cca99e51cb09742c444a8d37ad5a688644..0fd92dcc7f57392a5d4eccef39c0d431b2d75eb2 100644
--- a/graph/edges/graph-edge.page-type.ts
+++ b/graph/edges/graph-edge.page-type.ts
@@ -37,7 +37,7 @@ export const graphEdge = {
     },
     {
       invariantKind: "departure",
-      statement: "One doing both says on each edge which it was.",
+      statement: "One doing both says on each edge which that edge was.",
     },
     {
       invariantKind: "departure",
