diff --git a/temper/temper-lib-scrollable-menu/scrollable-menu-library-shapes/scrollable-menu-library-shapes.type-declaration.d.ts b/temper/temper-lib-scrollable-menu/scrollable-menu-library-shapes/scrollable-menu-library-shapes.type-declaration.d.ts
index 8b8aec1bf453dea6178c110b13cd8b51b4546fdb..3859b79cc53613897192c64226f6f05518baacbf 100644
--- a/temper/temper-lib-scrollable-menu/scrollable-menu-library-shapes/scrollable-menu-library-shapes.type-declaration.d.ts
+++ b/temper/temper-lib-scrollable-menu/scrollable-menu-library-shapes/scrollable-menu-library-shapes.type-declaration.d.ts
@@ -70,9 +70,7 @@ interface LsmComboBoxOptions {
 }
 
 interface ComboBoxBase extends ZoComboBoxInstance {
-  owner?: unknown
   m_comboBox?: unknown
-  options?: LsmComboBoxOptions
   updatedOptions?: Record<string, unknown>
   isContextMenu?: boolean
 }
