diff --git a/collection-system/royal-road-accounts/pages/alan/alan.royal-road-account.ts b/collection-system/royal-road-accounts/pages/alan/alan.royal-road-account.ts
index 88edf359f3796b300e441aacbf47fbf707d6ca97..9d086f9a6a000d15188f2ded915e53e231f35f19 100644
--- a/collection-system/royal-road-accounts/pages/alan/alan.royal-road-account.ts
+++ b/collection-system/royal-road-accounts/pages/alan/alan.royal-road-account.ts
@@ -4,5 +4,5 @@ export const alan = {
   id: "01a01257-3e4f-7000-a256-37b02dd285cb",
   pageTypeSlug: "royal-road-account",
   slug: "alan",
-  username: "aawalton@gmail.com",
+  email: "aawalton@gmail.com",
 } as const satisfies RoyalRoadAccount
diff --git a/collection-system/royal-road-accounts/properties/royal-road-email.email-address-property.ts b/collection-system/royal-road-accounts/properties/royal-road-email.email-address-property.ts
index cf424b51d6561ef4941bcea0f3c09b4aadf101f6..8a7ff517f083bd84ffef9edda8aa96cfbcdb2003 100644
--- a/collection-system/royal-road-accounts/properties/royal-road-email.email-address-property.ts
+++ b/collection-system/royal-road-accounts/properties/royal-road-email.email-address-property.ts
@@ -1,11 +1,11 @@
 import type { EmailAddressProperty } from "@akasha/pages-system/email-address-property"
 
-export type RoyalRoadUsername = string
+export type RoyalRoadEmail = string
 
 export const royalRoadEmail = {
   id: "01a0685d-b81f-79bb-a097-70cc1e0610d2",
   pageTypeSlug: "email-address-property",
   slug: "royal-road-email",
-  propertySlug: "username",
+  propertySlug: "email",
   definition: "the address a Royal Road account signs in as",
 } as const satisfies EmailAddressProperty
diff --git a/collection-system/royal-road-accounts/royal-road-account.page-type.ts b/collection-system/royal-road-accounts/royal-road-account.page-type.ts
index 5a8492ef40cb7fece18fc4d2a8365593492a3a98..a7bcb9e266eb9ad1ea6184558f8e136f4067e04c 100644
--- a/collection-system/royal-road-accounts/royal-road-account.page-type.ts
+++ b/collection-system/royal-road-accounts/royal-road-account.page-type.ts
@@ -1,10 +1,10 @@
 import type { Page } from "@akasha/pages-system/page"
 import type { PageType } from "@akasha/pages-system/page-type"
-import type { RoyalRoadUsername } from "./properties/royal-road-email.email-address-property.ts"
+import type { RoyalRoadEmail } from "./properties/royal-road-email.email-address-property.ts"
 import type { RoyalRoadPassword } from "./properties/royal-road-password.text-property.ts"
 
 export type RoyalRoadAccount = Page & {
-  username: RoyalRoadUsername
+  email: RoyalRoadEmail
   password?: RoyalRoadPassword
 }
 
