diff --git a/temper/temper-eso-types/eso-interface-extra-2/eso-interface-extra-2.type-declaration.d.ts b/temper/temper-eso-types/eso-interface-extra-2/eso-interface-extra-2.type-declaration.d.ts
index bf6c5f5075d8907ea214382d17905e2aeae0a16f..cc9149faa86132d89f1057b6360c4158d6567fb5 100644
--- a/temper/temper-eso-types/eso-interface-extra-2/eso-interface-extra-2.type-declaration.d.ts
+++ b/temper/temper-eso-types/eso-interface-extra-2/eso-interface-extra-2.type-declaration.d.ts
@@ -219,10 +219,6 @@ declare const zo_iconTextFormatNoSpace: (
   inheritColor?: boolean
 ) => string
 
-interface EndInWorldInteractionsFragment {
-  OnShown: (this: EndInWorldInteractionsFragment) => undefined
-}
-
 declare const TREASURE_MAP_INVENTORY_SCENE: Scene
 
 declare const FRAME_TARGET_CENTERED_FRAGMENT: SceneFragment
diff --git a/temper/temper-eso-types/eso-interface-extra/eso-interface-extra.type-declaration.d.ts b/temper/temper-eso-types/eso-interface-extra/eso-interface-extra.type-declaration.d.ts
index 2b099e5ebe3b09f67479cbc08d0916d71dd11e3e..36a0cab56f951ecf300ca7df19caa54d0ea373c1 100644
--- a/temper/temper-eso-types/eso-interface-extra/eso-interface-extra.type-declaration.d.ts
+++ b/temper/temper-eso-types/eso-interface-extra/eso-interface-extra.type-declaration.d.ts
@@ -90,10 +90,6 @@ interface ErrorFrameSingleton {
 
 declare const ZO_ERROR_FRAME: ErrorFrameSingleton
 
-interface MailInbox {
-  mailId?: Id64
-}
-
 declare const ZO_LeaderboardScoreProvider: object
 declare const ZO_GuildMotDProvider: object
 declare const ZO_CraftBagAutoTransferProvider: object
diff --git a/temper/temper-eso-types/eso-ui/eso-ui.type-declaration.d.ts b/temper/temper-eso-types/eso-ui/eso-ui.type-declaration.d.ts
index bbf3cb3565e9d1e2e82e92428b6b044474f3d4c3..579a2cdb764877da1dc2d6468d8c07581d732533 100644
--- a/temper/temper-eso-types/eso-ui/eso-ui.type-declaration.d.ts
+++ b/temper/temper-eso-types/eso-ui/eso-ui.type-declaration.d.ts
@@ -103,7 +103,6 @@ interface TopLevelWindow extends Control {
   GetTop: () => number
   StartMoving: () => void
   StopMovingOrResizing: () => void
-  BringWindowToTop: () => void
 }
 
 type CtControl = number & { readonly __ct: "CT_CONTROL" }
