diff --git a/command-system/commands/move/repointing/move-repointing.module.test.ts b/command-system/commands/move/repointing/move-repointing.module.test.ts
index aa67bf93394d5646d89eb8e079030762f85772d9..5d6e769d37eb3b6a53cd6ce7dc768e8292b61051 100644
--- a/command-system/commands/move/repointing/move-repointing.module.test.ts
+++ b/command-system/commands/move/repointing/move-repointing.module.test.ts
@@ -19,14 +19,14 @@ const TABLE = "akasha/one/routes.ts"
 
 const TYPED_ROUTE = "akasha/one/routes/api.addons.download.ts"
 
-const TYPED_ROUTE_AT = "akasha/one/routes/addon-bundle/addon-bundle.route.code.ts"
+const TYPED_ROUTE_AT = "akasha/one/routes/addon-parcel/addon-parcel.route.code.ts"
 
 const TYPED_CODE = `import type { Route } from "./+types/api.addons.download"
 
 export const it: Route | null = null
 `
 
-const TYPED_REPOINTED = `import type { Route } from "./+types/addon-bundle.route.code"
+const TYPED_REPOINTED = `import type { Route } from "./+types/addon-parcel.route.code"
 
 export const it: Route | null = null
 `
